<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>C Vignette de zazou | Prise en compte de l’organisation hiérarchique des espèces pour la découverte de signatures métagénomiques multi-échelles</title>
  <meta name="description" content="Thèse de mathématiques appliquées par Antoine Bichat. Statistique – Apprentissage – Métagénomique – Arbre phylogénétique – Tests multiples – Processus stochastiques" />
  <meta name="generator" content="bookdown 0.21 and GitBook 2.6.7" />

  <meta property="og:title" content="C Vignette de zazou | Prise en compte de l’organisation hiérarchique des espèces pour la découverte de signatures métagénomiques multi-échelles" />
  <meta property="og:type" content="book" />
  <meta property="og:url" content="https://abichat.github.io/thesis/" />
  <meta property="og:image" content="https://abichat.github.io/thesis/img/tree_ou.png" />
  <meta property="og:description" content="Thèse de mathématiques appliquées par Antoine Bichat. Statistique – Apprentissage – Métagénomique – Arbre phylogénétique – Tests multiples – Processus stochastiques" />
  <meta name="github-repo" content="abichat/thesis" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="C Vignette de zazou | Prise en compte de l’organisation hiérarchique des espèces pour la découverte de signatures métagénomiques multi-échelles" />
  
  <meta name="twitter:description" content="Thèse de mathématiques appliquées par Antoine Bichat. Statistique – Apprentissage – Métagénomique – Arbre phylogénétique – Tests multiples – Processus stochastiques" />
  <meta name="twitter:image" content="https://abichat.github.io/thesis/img/tree_ou.png" />




  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  <link rel="shortcut icon" href="img/favicon.ico" type="image/x-icon" />
<link rel="prev" href="B-productions-scientifiques.html"/>
<link rel="next" href="références.html"/>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<script src="libs/accessible-code-block-0.0.1/empty-anchor.js"></script>
<script>
(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
})(window,document,'script','https://www.google-analytics.com/analytics.js','ga');

ga('create', 'UA-131161788-01', 'auto');
ga('send', 'pageview');
</script>


<style type="text/css">
code.sourceCode > span { display: inline-block; line-height: 1.25; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode { white-space: pre; position: relative; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
code.sourceCode { white-space: pre-wrap; }
code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<br><br><br>
<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Introduction</a></li>
<li class="chapter" data-level="1" data-path="1-metagenomique.html"><a href="1-metagenomique.html"><i class="fa fa-check"></i><b>1</b> Métagénomique</a><ul>
<li class="chapter" data-level="1.1" data-path="1-metagenomique.html"><a href="1-metagenomique.html#microbiote"><i class="fa fa-check"></i><b>1.1</b> Le microbiote intestinal humain</a><ul>
<li class="chapter" data-level="1.1.1" data-path="1-metagenomique.html"><a href="1-metagenomique.html#description"><i class="fa fa-check"></i><b>1.1.1</b> Description</a></li>
<li class="chapter" data-level="1.1.2" data-path="1-metagenomique.html"><a href="1-metagenomique.html#rôle"><i class="fa fa-check"></i><b>1.1.2</b> Rôle</a></li>
<li class="chapter" data-level="1.1.3" data-path="1-metagenomique.html"><a href="1-metagenomique.html#dysbioses"><i class="fa fa-check"></i><b>1.1.3</b> Dysbioses</a></li>
<li class="chapter" data-level="1.1.4" data-path="1-metagenomique.html"><a href="1-metagenomique.html#utilisations"><i class="fa fa-check"></i><b>1.1.4</b> Utilisations</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="1-metagenomique.html"><a href="1-metagenomique.html#caracterisation"><i class="fa fa-check"></i><b>1.2</b> Caractérisation du microbiote</a><ul>
<li class="chapter" data-level="1.2.1" data-path="1-metagenomique.html"><a href="1-metagenomique.html#collecte-des-échantillons"><i class="fa fa-check"></i><b>1.2.1</b> Collecte des échantillons</a></li>
<li class="chapter" data-level="1.2.2" data-path="1-metagenomique.html"><a href="1-metagenomique.html#séquençage-par-gène-marqueur"><i class="fa fa-check"></i><b>1.2.2</b> Séquençage par gène marqueur</a></li>
<li class="chapter" data-level="1.2.3" data-path="1-metagenomique.html"><a href="1-metagenomique.html#séquençage-non-ciblé"><i class="fa fa-check"></i><b>1.2.3</b> Séquençage non ciblé</a></li>
<li class="chapter" data-level="1.2.4" data-path="1-metagenomique.html"><a href="1-metagenomique.html#donneesmetagenomiques"><i class="fa fa-check"></i><b>1.2.4</b> Données métagénomiques</a></li>
</ul></li>
<li class="chapter" data-level="1.3" data-path="1-metagenomique.html"><a href="1-metagenomique.html#jeuxdonnees"><i class="fa fa-check"></i><b>1.3</b> Jeux de données</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="2-anadiff.html"><a href="2-anadiff.html"><i class="fa fa-check"></i><b>2</b> Études d’analyse différentielle</a><ul>
<li class="chapter" data-level="2.1" data-path="2-anadiff.html"><a href="2-anadiff.html#tests-statistiques"><i class="fa fa-check"></i><b>2.1</b> Tests statistiques</a><ul>
<li class="chapter" data-level="2.1.1" data-path="2-anadiff.html"><a href="2-anadiff.html#analyse-de-la-variance-à-un-facteur"><i class="fa fa-check"></i><b>2.1.1</b> Analyse de la variance à un facteur</a></li>
<li class="chapter" data-level="2.1.2" data-path="2-anadiff.html"><a href="2-anadiff.html#test-de-wilcoxon"><i class="fa fa-check"></i><b>2.1.2</b> Test de Wilcoxon</a></li>
<li class="chapter" data-level="2.1.3" data-path="2-anadiff.html"><a href="2-anadiff.html#test-de-kruskall-wallis"><i class="fa fa-check"></i><b>2.1.3</b> Test de Kruskall-Wallis</a></li>
<li class="chapter" data-level="2.1.4" data-path="2-anadiff.html"><a href="2-anadiff.html#autres-methodes"><i class="fa fa-check"></i><b>2.1.4</b> Autres méthodes</a></li>
</ul></li>
<li class="chapter" data-level="2.2" data-path="2-anadiff.html"><a href="2-anadiff.html#problématique-des-tests-multiples"><i class="fa fa-check"></i><b>2.2</b> Problématique des tests multiples</a><ul>
<li class="chapter" data-level="2.2.1" data-path="2-anadiff.html"><a href="2-anadiff.html#évaluation-des-performances"><i class="fa fa-check"></i><b>2.2.1</b> Évaluation des performances</a></li>
<li class="chapter" data-level="2.2.2" data-path="2-anadiff.html"><a href="2-anadiff.html#correction-de-bonferroni"><i class="fa fa-check"></i><b>2.2.2</b> Correction de Bonferroni</a></li>
<li class="chapter" data-level="2.2.3" data-path="2-anadiff.html"><a href="2-anadiff.html#correction-de-benjamini-hochberg"><i class="fa fa-check"></i><b>2.2.3</b> Correction de Benjamini-Hochberg</a></li>
<li class="chapter" data-level="2.2.4" data-path="2-anadiff.html"><a href="2-anadiff.html#correction-de-benjamini-yekutieli"><i class="fa fa-check"></i><b>2.2.4</b> Correction de Benjamini-Yekutieli</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="2-anadiff.html"><a href="2-anadiff.html#procédures-hiérarchiques-pour-tests-multiples"><i class="fa fa-check"></i><b>2.3</b> Procédures hiérarchiques pour tests multiples</a><ul>
<li class="chapter" data-level="2.3.1" data-path="2-anadiff.html"><a href="2-anadiff.html#treefdr"><i class="fa fa-check"></i><b>2.3.1</b> <em>TreeFDR</em></a></li>
<li class="chapter" data-level="2.3.2" data-path="2-anadiff.html"><a href="2-anadiff.html#fdr-hiérarchique"><i class="fa fa-check"></i><b>2.3.2</b> FDR hiérarchique</a></li>
<li class="chapter" data-level="2.3.3" data-path="2-anadiff.html"><a href="2-anadiff.html#treeclimbr"><i class="fa fa-check"></i><b>2.3.3</b> <em>treeclimbR</em></a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="3" data-path="3-arbres.html"><a href="3-arbres.html"><i class="fa fa-check"></i><b>3</b> Arbres</a><ul>
<li class="chapter" data-level="3.1" data-path="3-arbres.html"><a href="3-arbres.html#définitions"><i class="fa fa-check"></i><b>3.1</b> Définitions</a></li>
<li class="chapter" data-level="3.2" data-path="3-arbres.html"><a href="3-arbres.html#distances"><i class="fa fa-check"></i><b>3.2</b> Distances</a><ul>
<li class="chapter" data-level="3.2.1" data-path="3-arbres.html"><a href="3-arbres.html#distance-de-robinson-foulds"><i class="fa fa-check"></i><b>3.2.1</b> Distance de Robinson-Foulds</a></li>
<li class="chapter" data-level="3.2.2" data-path="3-arbres.html"><a href="3-arbres.html#distance-cophénétique"><i class="fa fa-check"></i><b>3.2.2</b> Distance cophénétique</a></li>
<li class="chapter" data-level="3.2.3" data-path="3-arbres.html"><a href="3-arbres.html#distance-de-billera-holmes-vogtmann-bhv"><i class="fa fa-check"></i><b>3.2.3</b> Distance de Billera-Holmes-Vogtmann (BHV)</a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="3-arbres.html"><a href="3-arbres.html#arbres-dintérêt"><i class="fa fa-check"></i><b>3.3</b> Arbres d’intérêt</a><ul>
<li class="chapter" data-level="3.3.1" data-path="3-arbres.html"><a href="3-arbres.html#phylogénie"><i class="fa fa-check"></i><b>3.3.1</b> Phylogénie</a></li>
<li class="chapter" data-level="3.3.2" data-path="3-arbres.html"><a href="3-arbres.html#taxonomie"><i class="fa fa-check"></i><b>3.3.2</b> Taxonomie</a></li>
<li class="chapter" data-level="3.3.3" data-path="3-arbres.html"><a href="3-arbres.html#cortree"><i class="fa fa-check"></i><b>3.3.3</b> Arbre des corrélations</a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="3-arbres.html"><a href="3-arbres.html#comparaison-entre-les-arbres"><i class="fa fa-check"></i><b>3.4</b> Comparaison entre les arbres</a><ul>
<li class="chapter" data-level="3.4.1" data-path="3-arbres.html"><a href="3-arbres.html#forêt-darbres"><i class="fa fa-check"></i><b>3.4.1</b> Forêt d’arbres</a></li>
<li class="chapter" data-level="3.4.2" data-path="3-arbres.html"><a href="3-arbres.html#distance-entre-les-arbres"><i class="fa fa-check"></i><b>3.4.2</b> Distance entre les arbres</a></li>
<li class="chapter" data-level="3.4.3" data-path="3-arbres.html"><a href="3-arbres.html#checktreefdr"><i class="fa fa-check"></i><b>3.4.3</b> Choix de l’arbre et lissage de z-score</a></li>
<li class="chapter" data-level="3.4.4" data-path="3-arbres.html"><a href="3-arbres.html#choix-de-larbre-et-fdr-hiérarchique"><i class="fa fa-check"></i><b>3.4.4</b> Choix de l’arbre et FDR hiérarchique</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="4-nouvelleapproche.html"><a href="4-nouvelleapproche.html"><i class="fa fa-check"></i><b>4</b> <em>zazou</em> : une nouvelle approche</a><ul>
<li class="chapter" data-level="4.1" data-path="4-nouvelleapproche.html"><a href="4-nouvelleapproche.html#ou"><i class="fa fa-check"></i><b>4.1</b> Processus d’Ornstein-Uhlenbeck</a></li>
<li class="chapter" data-level="4.2" data-path="4-nouvelleapproche.html"><a href="4-nouvelleapproche.html#zazousection"><i class="fa fa-check"></i><b>4.2</b> Zazou</a><ul>
<li class="chapter" data-level="4.2.1" data-path="4-nouvelleapproche.html"><a href="4-nouvelleapproche.html#modèle"><i class="fa fa-check"></i><b>4.2.1</b> Modèle</a></li>
<li class="chapter" data-level="4.2.2" data-path="4-nouvelleapproche.html"><a href="4-nouvelleapproche.html#estimation-ponctuelle"><i class="fa fa-check"></i><b>4.2.2</b> Estimation ponctuelle</a></li>
<li class="chapter" data-level="4.2.3" data-path="4-nouvelleapproche.html"><a href="4-nouvelleapproche.html#débiaisage-et-intervalles-de-confiance"><i class="fa fa-check"></i><b>4.2.3</b> Débiaisage et intervalles de confiance</a></li>
<li class="chapter" data-level="4.2.4" data-path="4-nouvelleapproche.html"><a href="4-nouvelleapproche.html#correction-pour-tests-multiples"><i class="fa fa-check"></i><b>4.2.4</b> Correction pour tests multiples</a></li>
</ul></li>
<li class="chapter" data-level="4.3" data-path="4-nouvelleapproche.html"><a href="4-nouvelleapproche.html#evaluation"><i class="fa fa-check"></i><b>4.3</b> Évaluation de la méthode</a><ul>
<li class="chapter" data-level="4.3.1" data-path="4-nouvelleapproche.html"><a href="4-nouvelleapproche.html#simuzazou"><i class="fa fa-check"></i><b>4.3.1</b> Données simulées</a></li>
<li class="chapter" data-level="4.3.2" data-path="4-nouvelleapproche.html"><a href="4-nouvelleapproche.html#influence-de-lâge"><i class="fa fa-check"></i><b>4.3.2</b> Influence de l’âge</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="5-ananum.html"><a href="5-ananum.html"><i class="fa fa-check"></i><b>5</b> Problèmes d’analyse numérique</a><ul>
<li class="chapter" data-level="5.1" data-path="5-ananum.html"><a href="5-ananum.html#shooting"><i class="fa fa-check"></i><b>5.1</b> Algorithme du <em>shooting</em></a></li>
<li class="chapter" data-level="5.2" data-path="5-ananum.html"><a href="5-ananum.html#mam"><i class="fa fa-check"></i><b>5.2</b> Minimisation sous contrainte de <span class="math inline">\(x^TAx\)</span></a></li>
<li class="chapter" data-level="5.3" data-path="5-ananum.html"><a href="5-ananum.html#proj"><i class="fa fa-check"></i><b>5.3</b> Projection sur un ensemble de faisabilité</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="conclusion-et-perspectives.html"><a href="conclusion-et-perspectives.html"><i class="fa fa-check"></i>Conclusion et perspectives</a><ul>
<li class="chapter" data-level="" data-path="conclusion-et-perspectives.html"><a href="conclusion-et-perspectives.html#association-versus-prédiction"><i class="fa fa-check"></i>Association versus prédiction</a></li>
<li class="chapter" data-level="" data-path="conclusion-et-perspectives.html"><a href="conclusion-et-perspectives.html#autres-procédures-hiérarchiques"><i class="fa fa-check"></i>Autres procédures hiérarchiques</a></li>
<li class="chapter" data-level="" data-path="conclusion-et-perspectives.html"><a href="conclusion-et-perspectives.html#amélioration-de-zazou"><i class="fa fa-check"></i>Amélioration de zazou</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="digest.html"><a href="digest.html"><i class="fa fa-check"></i>Digest</a><ul>
<li class="chapter" data-level="" data-path="digest.html"><a href="digest.html#chapter-i"><i class="fa fa-check"></i>Chapter I</a></li>
<li class="chapter" data-level="" data-path="digest.html"><a href="digest.html#chapter-ii"><i class="fa fa-check"></i>Chapter II</a></li>
<li class="chapter" data-level="" data-path="digest.html"><a href="digest.html#chapter-iii"><i class="fa fa-check"></i>Chapter III</a></li>
<li class="chapter" data-level="" data-path="digest.html"><a href="digest.html#chapter-iv"><i class="fa fa-check"></i>Chapter IV</a></li>
<li class="chapter" data-level="" data-path="digest.html"><a href="digest.html#chapter-v"><i class="fa fa-check"></i>Chapter V</a></li>
<li class="chapter" data-level="" data-path="digest.html"><a href="digest.html#conclusions-and-outlooks"><i class="fa fa-check"></i>Conclusions and outlooks</a></li>
</ul></li>
<li class="appendix"><span><b>Annexes</b></span></li>
<li class="chapter" data-level="A" data-path="A-notations.html"><a href="A-notations.html"><i class="fa fa-check"></i><b>A</b> Notations</a></li>
<li class="chapter" data-level="B" data-path="B-productions-scientifiques.html"><a href="B-productions-scientifiques.html"><i class="fa fa-check"></i><b>B</b> Productions scientifiques</a><ul>
<li class="chapter" data-level="" data-path="B-productions-scientifiques.html"><a href="B-productions-scientifiques.html#quantifying-the-impact-of-tree-choice-in-metagenomics-differential-abundance-studies-with-r"><i class="fa fa-check"></i>Quantifying the impact of tree choice in metagenomics differential abundance studies with R</a></li>
<li class="chapter" data-level="" data-path="B-productions-scientifiques.html"><a href="B-productions-scientifiques.html#incorporating-phylogenetic-information-in-microbiome-differential-abundance-studies-has-no-effect-on-detection-power-and-fdr-control"><i class="fa fa-check"></i>Incorporating Phylogenetic Information in Microbiome Differential Abundance Studies Has No Effect on Detection Power and FDR Control</a></li>
<li class="chapter" data-level="" data-path="B-productions-scientifiques.html"><a href="B-productions-scientifiques.html#hierarchical-correction-of-p-values-via-a-tree-running-ornstein-uhlenbeck-process"><i class="fa fa-check"></i>Hierarchical correction of p-values via a tree running Ornstein-Uhlenbeck process</a></li>
<li class="chapter" data-level="" data-path="B-productions-scientifiques.html"><a href="B-productions-scientifiques.html#packages-r"><i class="fa fa-check"></i>Packages R</a><ul>
<li class="chapter" data-level="" data-path="B-productions-scientifiques.html"><a href="B-productions-scientifiques.html#yatah"><i class="fa fa-check"></i>yatah</a></li>
<li class="chapter" data-level="" data-path="B-productions-scientifiques.html"><a href="B-productions-scientifiques.html#evabic"><i class="fa fa-check"></i>evabic</a></li>
<li class="chapter" data-level="" data-path="B-productions-scientifiques.html"><a href="B-productions-scientifiques.html#correlationtree"><i class="fa fa-check"></i>correlationtree</a></li>
<li class="chapter" data-level="" data-path="B-productions-scientifiques.html"><a href="B-productions-scientifiques.html#zazou"><i class="fa fa-check"></i>zazou</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="C" data-path="C-vignette.html"><a href="C-vignette.html"><i class="fa fa-check"></i><b>C</b> Vignette de zazou</a></li>
<li class="chapter" data-level="" data-path="références.html"><a href="références.html"><i class="fa fa-check"></i>Références</a></li>
<li class="chapter" data-level="" data-path="mentions-légales.html"><a href="mentions-légales.html"><i class="fa fa-check"></i>Mentions légales</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Prise en compte de l’organisation hiérarchique des espèces pour la découverte de signatures métagénomiques multi-échelles</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<script type="text/x-mathjax-config">
MathJax.Hub.Config({
  TeX: {
    Macros: {
      NN: "{\\mathbb{N}}",
      ZZ: "{\\mathbb{Z}}",
      QQ: "{\\mathbb{Q}}",
      RR: "{\\mathbb{R}}",
      shiftset: "{\\mathcal{D}}",
      dx: ["{\\mathrm{d}^{#1}\\mspace{-1mu}\\mathord{#2}}", 2, ""],
      indic: "{\\unicode{x1D7D9}}",
      prob: "\\mathop{\\mathbb{P}}",
      esp: "\\mathop{\\mathbb{E}}",
      var: "\\mathop{\\mathbb{V}\\text{ar}}",
      cov: "\\mathop{\\mathbb{C}\\text{ov}}",
      PP: ["{\\prob\\left({#1}\\right)}", 1],
      EE: ["{\\esp\\left[{#1}\\right]}", 1],
      VV: ["{\\var\\left[{#1}\\right]}", 1],
      CC: ["{\\cov\\left[{#1}\\right]}", 1],
      normal: ["{\\mathcal{N}\\left({#1},{#2}\\right)}", 2],
      ou: ["{#1}_{\\text{ou}}", 1],
      oui: ["{#1}_{\\text{ou},#2}", 2],
      pv: "{\\mathfrak{p}}",
      qv: "{\\mathfrak{q}}",
      zs: "{\\mathfrak{z}}",
      ts: "{\\mathfrak{t}}",
      sign: "{\\mathfrak{s}}",
      shifts: "{\\delta}",
      optim: "{\\beta}",
      param: "{\\theta}",
      unif: ["{\\mathcal{U}\\left({#1}\\right)}", 1],
      argmin: "\\mathop{\\mathrm{argmin}}",
      diag: "\\mathop{\\mathrm{Diag}}",
      rang: "\\mathop{\\mathrm{rang}}",
      pa: "\\mathop{\\mathrm{pa}}",
      mrca: "\\mathop{\\mathrm{mrca}}",
      desc: "\\mathop{\\mathrm{desc}}",
      warning: ["\\color{red}{{#1}}", 1]
    }
  }
});
</script>
<div id="vignette" class="section level1">
<h1><span class="header-section-number">C</span> Vignette de zazou</h1>
<p>Nous commençons par charger les <em>packages</em> qui seront utilisés dans cette vignette.</p>
<div class="sourceCode" id="cb1"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1-1"><a href="C-vignette.html#cb1-1"></a><span class="kw">library</span>(zazou)</span>
<span id="cb1-2"><a href="C-vignette.html#cb1-2"></a><span class="kw">library</span>(evabic)</span>
<span id="cb1-3"><a href="C-vignette.html#cb1-3"></a><span class="kw">library</span>(dplyr)</span>
<span id="cb1-4"><a href="C-vignette.html#cb1-4"></a><span class="kw">library</span>(ggplot2)</span>
<span id="cb1-5"><a href="C-vignette.html#cb1-5"></a><span class="kw">set.seed</span>(<span class="dv">20201209</span>)</span></code></pre></div>
<p>On récupère ensuite le jeu de données de <span class="citation">Wu et al. (<a href="références.html#ref-wu2011linking" role="doc-biblioref">2011</a>)</span>, où l’on ne conserve que les individus avec une faible consommation d’alcool. On modifie les longueurs de branches de la phylogénie pour que celle-ci soit ultramétrique.</p>
<div class="sourceCode" id="cb2"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb2-1"><a href="C-vignette.html#cb2-1"></a><span class="kw">data</span>(<span class="st">&quot;alcohol&quot;</span>)</span>
<span id="cb2-2"><a href="C-vignette.html#cb2-2"></a>abund &lt;-<span class="st"> </span>alcohol<span class="op">$</span>X[, alcohol<span class="op">$</span>Y <span class="op">==</span><span class="st"> &quot;Low&quot;</span>]</span>
<span id="cb2-3"><a href="C-vignette.html#cb2-3"></a>tree &lt;-<span class="st"> </span><span class="kw">force_ultrametric</span>(alcohol<span class="op">$</span>tree)</span></code></pre></div>
<p>On filtre ensuite les OTUs qui sont présentes dans au moins 25 échantillons parmi les 49.</p>
<div class="sourceCode" id="cb3"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb3-1"><a href="C-vignette.html#cb3-1"></a>abund &lt;-<span class="st"> </span>abund[<span class="kw">rowSums</span>(abund <span class="op">&gt;</span><span class="st"> </span><span class="dv">0</span>) <span class="op">&gt;</span><span class="st"> </span><span class="dv">25</span>, ]</span>
<span id="cb3-2"><a href="C-vignette.html#cb3-2"></a>tree &lt;-<span class="st"> </span>ape<span class="op">::</span><span class="kw">drop.tip</span>(tree, <span class="kw">setdiff</span>(tree<span class="op">$</span>tip.label, <span class="kw">rownames</span>(abund)))</span></code></pre></div>
<p>Il en reste alors 91.</p>
<p>Nous assignons ensuite chaque échantillon aléatoirement à un groupe <span class="math inline">\(A\)</span> ou <span class="math inline">\(B\)</span>.</p>
<div class="sourceCode" id="cb4"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb4-1"><a href="C-vignette.html#cb4-1"></a>groups &lt;-<span class="st"> </span><span class="kw">sample</span>(<span class="kw">c</span>(<span class="st">&quot;A&quot;</span>, <span class="st">&quot;B&quot;</span>), <span class="dt">size =</span> <span class="kw">ncol</span>(abund), <span class="dt">replace =</span> <span class="ot">TRUE</span>)</span>
<span id="cb4-2"><a href="C-vignette.html#cb4-2"></a>pvalues_original &lt;-<span class="st"> </span><span class="kw">test_wilcoxon</span>(abund, groups)<span class="op">$</span>p.value</span>
<span id="cb4-3"><a href="C-vignette.html#cb4-3"></a>zscores_original &lt;-<span class="st"> </span><span class="kw">p2z</span>(pvalues_original)</span>
<span id="cb4-4"><a href="C-vignette.html#cb4-4"></a><span class="kw">sum</span>(pvalues_original <span class="op">&lt;</span><span class="st"> </span><span class="fl">0.05</span>)</span></code></pre></div>
<pre><code>[1] 7</code></pre>
<p>Certaines OTUs sont déjà considérées comme différentiellement abondantes.</p>
<p>L’ensemble des OTUs est partitionné en <span class="math inline">\(20\)</span> groupes cohérents avec la phylogénie, comme expliqué dans la section <a href="4-nouvelleapproche.html#simuzazou">4.3.1</a>. Puis <span class="math inline">\(5\)</span> de ces groupes et les OTUs qu’ils contiennent sont tirés aléatoirement pour être différentiellement abondants.</p>
<div class="sourceCode" id="cb6"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb6-1"><a href="C-vignette.html#cb6-1"></a>clustering &lt;-<span class="st"> </span><span class="kw">create_clusters</span>(tree, <span class="dt">N_clusters =</span> <span class="dv">20</span>, </span>
<span id="cb6-2"><a href="C-vignette.html#cb6-2"></a>                              <span class="dt">method =</span> <span class="st">&quot;paraphyletic&quot;</span>)</span>
<span id="cb6-3"><a href="C-vignette.html#cb6-3"></a>clusters_da &lt;-<span class="st"> </span><span class="kw">sample</span>(<span class="dv">20</span>, <span class="dv">5</span>)</span>
<span id="cb6-4"><a href="C-vignette.html#cb6-4"></a>clusters_da</span></code></pre></div>
<pre><code>[1]  5  4  6  1 17</code></pre>
<div class="sourceCode" id="cb8"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb8-1"><a href="C-vignette.html#cb8-1"></a>otus_da &lt;-<span class="st"> </span><span class="kw">names</span>(clustering[<span class="kw">which</span>(clustering <span class="op">%in%</span><span class="st"> </span>clusters_da)])</span>
<span id="cb8-2"><a href="C-vignette.html#cb8-2"></a><span class="dv">1</span> <span class="op">-</span><span class="st"> </span><span class="kw">length</span>(otus_da) <span class="op">/</span><span class="st"> </span><span class="kw">nrow</span>(abund)</span></code></pre></div>
<pre><code>[1] 0.6483516</code></pre>
<p>On applique ensuite un <em>fold-change</em> de <span class="math inline">\(10\)</span> aux OTUs différentiellement abondants dans le groupe <span class="math inline">\(B\)</span>.</p>
<div class="sourceCode" id="cb10"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb10-1"><a href="C-vignette.html#cb10-1"></a>abund[otus_da, groups <span class="op">==</span><span class="st"> &quot;B&quot;</span>] &lt;-<span class="st"> </span><span class="dv">10</span> <span class="op">*</span><span class="st"> </span>abund[otus_da,  groups <span class="op">==</span><span class="st"> &quot;B&quot;</span>]</span></code></pre></div>
<p>On effectue un test de Wilcoxon sur ce nouveau jeu de données.</p>
<div class="sourceCode" id="cb11"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb11-1"><a href="C-vignette.html#cb11-1"></a>pvalues &lt;-<span class="st"> </span><span class="kw">test_wilcoxon</span>(abund, groups)<span class="op">$</span>p.value</span>
<span id="cb11-2"><a href="C-vignette.html#cb11-2"></a>zscores &lt;-<span class="st"> </span><span class="kw">p2z</span>(pvalues)</span>
<span id="cb11-3"><a href="C-vignette.html#cb11-3"></a><span class="kw">plot_shifts</span>(tree, <span class="ot">NA</span>, <span class="dt">obs_scores =</span> zscores,</span>
<span id="cb11-4"><a href="C-vignette.html#cb11-4"></a>            <span class="dt">sup_scores =</span> <span class="kw">list</span>(</span>
<span id="cb11-5"><a href="C-vignette.html#cb11-5"></a>              <span class="kw">list</span>(<span class="dt">scores =</span> clustering,</span>
<span id="cb11-6"><a href="C-vignette.html#cb11-6"></a>                   <span class="dt">title =</span> <span class="st">&quot;Clusters&quot;</span>,</span>
<span id="cb11-7"><a href="C-vignette.html#cb11-7"></a>                   <span class="dt">color =</span> <span class="kw">as.character</span>(clustering)),</span>
<span id="cb11-8"><a href="C-vignette.html#cb11-8"></a>              <span class="kw">list</span>(<span class="dt">scores =</span> zscores <span class="op">-</span><span class="st"> </span>zscores_original,</span>
<span id="cb11-9"><a href="C-vignette.html#cb11-9"></a>                   <span class="dt">title =</span> <span class="st">&quot;Difference in z-scores after fold-change&quot;</span>,</span>
<span id="cb11-10"><a href="C-vignette.html#cb11-10"></a>                   <span class="dt">color =</span> <span class="kw">as.character</span>(<span class="kw">sign</span>(zscores <span class="op">-</span><span class="st"> </span>zscores_original)))</span>
<span id="cb11-11"><a href="C-vignette.html#cb11-11"></a>              ))</span></code></pre></div>
<pre><code>Warning: Removed 181 rows containing missing values (geom_label).</code></pre>
<p><img src="thesis_files/figure-html/unnamed-chunk-9-1.png" width="90%" style="display: block; margin: auto;" /></p>
<p>Nous donnons ensuite nos <span class="math inline">\(z\)</span>-scores à la fonction <code>estimate_shifts</code> pour déterminer la position idéale des sauts avec une régression lasso. La grille des <span class="math inline">\(\ou{\alpha}\)</span> est également spécifiée mais celle des <span class="math inline">\(\lambda\)</span> est déterminée automatiquement.</p>
<div class="sourceCode" id="cb13"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb13-1"><a href="C-vignette.html#cb13-1"></a>estimation_lasso &lt;-<span class="st"> </span><span class="kw">estimate_shifts</span>(<span class="dt">zscores =</span> zscores, <span class="dt">tree =</span> tree, </span>
<span id="cb13-2"><a href="C-vignette.html#cb13-2"></a>                                    <span class="dt">alphaOU =</span> <span class="kw">c</span>(<span class="fl">0.2</span>, <span class="fl">0.5</span>, <span class="dv">1</span>, <span class="dv">2</span>, <span class="dv">5</span>),</span>
<span id="cb13-3"><a href="C-vignette.html#cb13-3"></a>                                    <span class="dt">method =</span> <span class="st">&quot;lasso&quot;</span>)</span>
<span id="cb13-4"><a href="C-vignette.html#cb13-4"></a>estimation_lasso</span></code></pre></div>
<pre><code>Tree is binary with 91 leafs and 180 branches
Covariance matrix has been estimated from an OU with alpha = 5 and sigma = 3.162 
---
Method: lasso with model selection
Regularization parameter: lambda = 14.068 
Objective value: 100.466
BIC: 333.88
pBIC: 372.38
---
Estimated shifts: 0 -0.217 0 0 0 0 -0.539 -0.943 0 0 ...
3 shifts have been identified (ie 98.3 % of sparsity)
A parsimonious solution would involve 3 shifts
---
Observed z-scores:  -2.474 -1.929 -1.451 -2.324 0.549 0.45 -2.397 0.636 -3.463 1.943 ...
Estimated z-scores:   -1.7   -1.7   -1.7   -1.7  -1.7 -1.7   -1.7  -1.7   -1.7  -1.7 ...
90 z-scores have been shifted (ie 1.1 % of sparsity)</code></pre>
<div class="sourceCode" id="cb15"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb15-1"><a href="C-vignette.html#cb15-1"></a><span class="kw">plot</span>(estimation_lasso)</span></code></pre></div>
<pre><code>Warning: Removed 178 rows containing missing values (geom_label).</code></pre>
<p><img src="thesis_files/figure-html/unnamed-chunk-10-1.png" width="90%" style="display: block; margin: auto;" /></p>
<p>Trois sauts sur l’arbre ont été détectés, sur des branches ayant beaucoup de descendants.</p>
<p>Si nous voulons avoir des <span class="math inline">\(p\)</span>-valeurs et des intervalles de confiance, il faut utiliser une régression <em>scaled lasso</em> avant d’appliquer la fonction <code>estimate_confint</code>.</p>
<div class="sourceCode" id="cb17"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb17-1"><a href="C-vignette.html#cb17-1"></a>estimation_scaledlasso &lt;-<span class="st"> </span></span>
<span id="cb17-2"><a href="C-vignette.html#cb17-2"></a><span class="st">  </span><span class="kw">estimate_shifts</span>(<span class="dt">zscores =</span> zscores,</span>
<span id="cb17-3"><a href="C-vignette.html#cb17-3"></a>                  <span class="dt">tree =</span> tree, <span class="dt">alphaOU =</span> <span class="kw">c</span>(<span class="fl">0.2</span>, <span class="fl">0.5</span>, <span class="dv">1</span>, <span class="dv">2</span>, <span class="dv">5</span>),</span>
<span id="cb17-4"><a href="C-vignette.html#cb17-4"></a>                  <span class="dt">method =</span> <span class="st">&quot;scaled lasso&quot;</span>)</span>
<span id="cb17-5"><a href="C-vignette.html#cb17-5"></a>estimation_scoresystem &lt;-<span class="st"> </span></span>
<span id="cb17-6"><a href="C-vignette.html#cb17-6"></a><span class="st">  </span><span class="kw">estimate_confint</span>(estimation_scaledlasso, </span>
<span id="cb17-7"><a href="C-vignette.html#cb17-7"></a>                   <span class="dt">method =</span> <span class="st">&quot;score system&quot;</span>)</span></code></pre></div>
<pre><code>t_star is not feasible, falling back to default value.</code></pre>
<div class="sourceCode" id="cb19"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb19-1"><a href="C-vignette.html#cb19-1"></a>estimation_scoresystem</span></code></pre></div>
<pre><code>Tree is binary with 91 leafs and 180 branches
Method: score system 
Confidence threshold: 0.05 
---
z-scores:
   leaf   estimate     lower      upper     pvalue    qvalue
1   283 -12.887241 -25.64441 -0.1300754 0.02385467 0.8941353
2  3494 -13.363635 -29.06041  2.3331434 0.04759456 1.0000000
3  3470 -12.803510 -28.47336  2.8663408 0.05463910 1.0000000
4  1661 -13.171378 -27.61457  1.2718194 0.03693836 1.0000000
5  4206  -7.310320 -21.70978  7.0891438 0.15985974 1.0000000
6 10038  -7.386608 -21.75859  6.9853783 0.15688556 1.0000000
  ...</code></pre>
<div class="sourceCode" id="cb21"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb21-1"><a href="C-vignette.html#cb21-1"></a><span class="kw">plot</span>(estimation_scoresystem)</span></code></pre></div>
<pre><code>Warning: Removed 1 rows containing missing values (geom_label).</code></pre>
<p><img src="thesis_files/figure-html/unnamed-chunk-11-1.png" width="90%" style="display: block; margin: auto;" /></p>
<div class="sourceCode" id="cb23"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb23-1"><a href="C-vignette.html#cb23-1"></a>pvalues_smoothed &lt;-<span class="st"> </span><span class="kw">pull_pvalues</span>(estimation_scoresystem)</span></code></pre></div>
<p>Après le débiaisage, les sauts dans les branches ne sont plus parcimonieux.</p>
<p>Nous allons regarder les performances de notre prédicteur via une courbe ROC.</p>
<div class="sourceCode" id="cb24"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb24-1"><a href="C-vignette.html#cb24-1"></a>df_measures_zazou &lt;-<span class="st"> </span></span>
<span id="cb24-2"><a href="C-vignette.html#cb24-2"></a><span class="st">  </span><span class="kw">ebc_tidy_by_threshold</span>(<span class="dt">detection_values =</span> pvalues_smoothed, </span>
<span id="cb24-3"><a href="C-vignette.html#cb24-3"></a>                        <span class="dt">true =</span> otus_da, <span class="dt">all =</span> <span class="kw">rownames</span>(abund))</span>
<span id="cb24-4"><a href="C-vignette.html#cb24-4"></a>df_measures_classic &lt;-<span class="st"> </span></span>
<span id="cb24-5"><a href="C-vignette.html#cb24-5"></a><span class="st">  </span><span class="kw">ebc_tidy_by_threshold</span>(<span class="dt">detection_values =</span> pvalues, </span>
<span id="cb24-6"><a href="C-vignette.html#cb24-6"></a>                        <span class="dt">true =</span> otus_da, <span class="dt">all =</span> <span class="kw">rownames</span>(abund))</span>
<span id="cb24-7"><a href="C-vignette.html#cb24-7"></a><span class="kw">bind_rows</span>(<span class="kw">mutate</span>(df_measures_zazou, <span class="dt">type =</span> <span class="st">&quot;zazou&quot;</span>),</span>
<span id="cb24-8"><a href="C-vignette.html#cb24-8"></a>          <span class="kw">mutate</span>(df_measures_classic, <span class="dt">type =</span> <span class="st">&quot;classic&quot;</span>)) <span class="op">%&gt;%</span><span class="st"> </span></span>
<span id="cb24-9"><a href="C-vignette.html#cb24-9"></a><span class="st">  </span><span class="kw">ggplot</span>() <span class="op">+</span></span>
<span id="cb24-10"><a href="C-vignette.html#cb24-10"></a><span class="st">  </span><span class="kw">aes</span>(<span class="dt">x =</span> FPR, <span class="dt">y =</span> TPR, <span class="dt">color =</span> type) <span class="op">+</span></span>
<span id="cb24-11"><a href="C-vignette.html#cb24-11"></a><span class="st">  </span><span class="kw">geom_line</span>() <span class="op">+</span></span>
<span id="cb24-12"><a href="C-vignette.html#cb24-12"></a><span class="st">  </span><span class="kw">theme_bw</span>()</span></code></pre></div>
<p><img src="thesis_files/figure-html/unnamed-chunk-12-1.png" width="90%" style="display: block; margin: auto;" /></p>
<p>La courbe ROC de <em>zazou</em> est au dessus de celle de la méthode standard. Et ceci se traduit en terme d’AUC :</p>
<div class="sourceCode" id="cb25"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb25-1"><a href="C-vignette.html#cb25-1"></a><span class="kw">ebc_AUC_from_measures</span>(df_measures_zazou)</span></code></pre></div>
<pre><code>[1] 0.9825212</code></pre>
<div class="sourceCode" id="cb27"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb27-1"><a href="C-vignette.html#cb27-1"></a><span class="kw">ebc_AUC_from_measures</span>(df_measures_classic)</span></code></pre></div>
<pre><code>[1] 0.8850636</code></pre>
<p>Version de <code>{zazou}</code> utilisée dans cette analyse :</p>
<div class="sourceCode" id="cb29"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb29-1"><a href="C-vignette.html#cb29-1"></a><span class="kw">packageVersion</span>(<span class="st">&quot;zazou&quot;</span>)</span></code></pre></div>
<pre><code>[1] &#39;0.0.1&#39;</code></pre>

<!--
The bib chunk below must go last in this document according to how R Markdown renders.  More info is at http://rmarkdown.rstudio.com/authoring_bibliographies_and_citations.html
-->

<!-- 
If you'd like to change the name of the bibliography to something else,
delete "References" and replace it.
-->
</div>
            </section>

          </div>
        </div>
      </div>
<a href="B-productions-scientifiques.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="références.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": true,
"facebook": false,
"twitter": false,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["twitter", "linkedin", "facebook", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "night",
"family": "serif",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": [["thesis.pdf", "PDF"]],
"toc": {
"collapse": "section"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
