<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Références | Prise en compte de l’organisation hiérarchique des espèces pour la découverte de signatures métagénomiques multi-échelles</title>
  <meta name="description" content="Thèse de mathématiques appliquées par Antoine Bichat. Statistique – Apprentissage – Métagénomique – Arbre phylogénétique – Tests multiples – Processus stochastiques" />
  <meta name="generator" content="bookdown 0.20 and GitBook 2.6.7" />

  <meta property="og:title" content="Références | Prise en compte de l’organisation hiérarchique des espèces pour la découverte de signatures métagénomiques multi-échelles" />
  <meta property="og:type" content="book" />
  <meta property="og:url" content="https://abichat.github.io/thesis/" />
  <meta property="og:image" content="https://abichat.github.io/thesis/img/tree_ou.png" />
  <meta property="og:description" content="Thèse de mathématiques appliquées par Antoine Bichat. Statistique – Apprentissage – Métagénomique – Arbre phylogénétique – Tests multiples – Processus stochastiques" />
  <meta name="github-repo" content="abichat/thesis" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Références | Prise en compte de l’organisation hiérarchique des espèces pour la découverte de signatures métagénomiques multi-échelles" />
  
  <meta name="twitter:description" content="Thèse de mathématiques appliquées par Antoine Bichat. Statistique – Apprentissage – Métagénomique – Arbre phylogénétique – Tests multiples – Processus stochastiques" />
  <meta name="twitter:image" content="https://abichat.github.io/thesis/img/tree_ou.png" />




  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  <link rel="shortcut icon" href="img/favicon.ico" type="image/x-icon" />
<link rel="prev" href="C-vignette.html"/>
<link rel="next" href="mentions-légales.html"/>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<script src="libs/accessible-code-block-0.0.1/empty-anchor.js"></script>
<script>
(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
})(window,document,'script','https://www.google-analytics.com/analytics.js','ga');

ga('create', 'UA-131161788-01', 'auto');
ga('send', 'pageview');
</script>


<style type="text/css">
code.sourceCode > span { display: inline-block; line-height: 1.25; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode { white-space: pre; position: relative; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
code.sourceCode { white-space: pre-wrap; }
code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<br><br><br>
<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Introduction</a></li>
<li class="chapter" data-level="1" data-path="1-metagenomique.html"><a href="1-metagenomique.html"><i class="fa fa-check"></i><b>1</b> Métagénomique</a><ul>
<li class="chapter" data-level="1.1" data-path="1-metagenomique.html"><a href="1-metagenomique.html#microbiote"><i class="fa fa-check"></i><b>1.1</b> Le microbiote intestinal humain</a><ul>
<li class="chapter" data-level="1.1.1" data-path="1-metagenomique.html"><a href="1-metagenomique.html#description"><i class="fa fa-check"></i><b>1.1.1</b> Description</a></li>
<li class="chapter" data-level="1.1.2" data-path="1-metagenomique.html"><a href="1-metagenomique.html#rôle"><i class="fa fa-check"></i><b>1.1.2</b> Rôle</a></li>
<li class="chapter" data-level="1.1.3" data-path="1-metagenomique.html"><a href="1-metagenomique.html#dysbioses"><i class="fa fa-check"></i><b>1.1.3</b> Dysbioses</a></li>
<li class="chapter" data-level="1.1.4" data-path="1-metagenomique.html"><a href="1-metagenomique.html#utilisations"><i class="fa fa-check"></i><b>1.1.4</b> Utilisations</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="1-metagenomique.html"><a href="1-metagenomique.html#caracterisation"><i class="fa fa-check"></i><b>1.2</b> Caractérisation du microbiote</a><ul>
<li class="chapter" data-level="1.2.1" data-path="1-metagenomique.html"><a href="1-metagenomique.html#collecte-des-échantillons"><i class="fa fa-check"></i><b>1.2.1</b> Collecte des échantillons</a></li>
<li class="chapter" data-level="1.2.2" data-path="1-metagenomique.html"><a href="1-metagenomique.html#séquençage-par-gène-marqueur"><i class="fa fa-check"></i><b>1.2.2</b> Séquençage par gène marqueur</a></li>
<li class="chapter" data-level="1.2.3" data-path="1-metagenomique.html"><a href="1-metagenomique.html#séquençage-non-ciblé"><i class="fa fa-check"></i><b>1.2.3</b> Séquençage non ciblé</a></li>
<li class="chapter" data-level="1.2.4" data-path="1-metagenomique.html"><a href="1-metagenomique.html#donneesmetagenomiques"><i class="fa fa-check"></i><b>1.2.4</b> Données métagénomiques</a></li>
</ul></li>
<li class="chapter" data-level="1.3" data-path="1-metagenomique.html"><a href="1-metagenomique.html#jeuxdonnees"><i class="fa fa-check"></i><b>1.3</b> Jeux de données</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="2-anadiff.html"><a href="2-anadiff.html"><i class="fa fa-check"></i><b>2</b> Études d’analyse différentielle</a><ul>
<li class="chapter" data-level="2.1" data-path="2-anadiff.html"><a href="2-anadiff.html#tests-statistiques"><i class="fa fa-check"></i><b>2.1</b> Tests statistiques</a><ul>
<li class="chapter" data-level="2.1.1" data-path="2-anadiff.html"><a href="2-anadiff.html#analyse-de-la-variance-à-un-facteur"><i class="fa fa-check"></i><b>2.1.1</b> Analyse de la variance à un facteur</a></li>
<li class="chapter" data-level="2.1.2" data-path="2-anadiff.html"><a href="2-anadiff.html#test-de-wilcoxon"><i class="fa fa-check"></i><b>2.1.2</b> Test de Wilcoxon</a></li>
<li class="chapter" data-level="2.1.3" data-path="2-anadiff.html"><a href="2-anadiff.html#test-de-kruskall-wallis"><i class="fa fa-check"></i><b>2.1.3</b> Test de Kruskall-Wallis</a></li>
<li class="chapter" data-level="2.1.4" data-path="2-anadiff.html"><a href="2-anadiff.html#autres-methodes"><i class="fa fa-check"></i><b>2.1.4</b> Autres méthodes</a></li>
</ul></li>
<li class="chapter" data-level="2.2" data-path="2-anadiff.html"><a href="2-anadiff.html#problématique-des-tests-multiples"><i class="fa fa-check"></i><b>2.2</b> Problématique des tests multiples</a><ul>
<li class="chapter" data-level="2.2.1" data-path="2-anadiff.html"><a href="2-anadiff.html#évaluation-des-performances"><i class="fa fa-check"></i><b>2.2.1</b> Évaluation des performances</a></li>
<li class="chapter" data-level="2.2.2" data-path="2-anadiff.html"><a href="2-anadiff.html#correction-de-bonferroni"><i class="fa fa-check"></i><b>2.2.2</b> Correction de Bonferroni</a></li>
<li class="chapter" data-level="2.2.3" data-path="2-anadiff.html"><a href="2-anadiff.html#correction-de-benjamini-hochberg"><i class="fa fa-check"></i><b>2.2.3</b> Correction de Benjamini-Hochberg</a></li>
<li class="chapter" data-level="2.2.4" data-path="2-anadiff.html"><a href="2-anadiff.html#correction-de-benjamini-yekutieli"><i class="fa fa-check"></i><b>2.2.4</b> Correction de Benjamini-Yekutieli</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="2-anadiff.html"><a href="2-anadiff.html#procédures-hiérarchiques-pour-tests-multiples"><i class="fa fa-check"></i><b>2.3</b> Procédures hiérarchiques pour tests multiples</a><ul>
<li class="chapter" data-level="2.3.1" data-path="2-anadiff.html"><a href="2-anadiff.html#treefdr"><i class="fa fa-check"></i><b>2.3.1</b> <em>TreeFDR</em></a></li>
<li class="chapter" data-level="2.3.2" data-path="2-anadiff.html"><a href="2-anadiff.html#fdr-hiérarchique"><i class="fa fa-check"></i><b>2.3.2</b> FDR hiérarchique</a></li>
<li class="chapter" data-level="2.3.3" data-path="2-anadiff.html"><a href="2-anadiff.html#treeclimbr"><i class="fa fa-check"></i><b>2.3.3</b> <em>treeclimbR</em></a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="3" data-path="3-arbres.html"><a href="3-arbres.html"><i class="fa fa-check"></i><b>3</b> Arbres</a><ul>
<li class="chapter" data-level="3.1" data-path="3-arbres.html"><a href="3-arbres.html#définitions"><i class="fa fa-check"></i><b>3.1</b> Définitions</a></li>
<li class="chapter" data-level="3.2" data-path="3-arbres.html"><a href="3-arbres.html#distances"><i class="fa fa-check"></i><b>3.2</b> Distances</a><ul>
<li class="chapter" data-level="3.2.1" data-path="3-arbres.html"><a href="3-arbres.html#distance-de-robinson-foulds"><i class="fa fa-check"></i><b>3.2.1</b> Distance de Robinson-Foulds</a></li>
<li class="chapter" data-level="3.2.2" data-path="3-arbres.html"><a href="3-arbres.html#distance-cophénétique"><i class="fa fa-check"></i><b>3.2.2</b> Distance cophénétique</a></li>
<li class="chapter" data-level="3.2.3" data-path="3-arbres.html"><a href="3-arbres.html#distance-de-billera-holmes-vogtmann-bhv"><i class="fa fa-check"></i><b>3.2.3</b> Distance de Billera-Holmes-Vogtmann (BHV)</a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="3-arbres.html"><a href="3-arbres.html#arbres-dintérêt"><i class="fa fa-check"></i><b>3.3</b> Arbres d’intérêt</a><ul>
<li class="chapter" data-level="3.3.1" data-path="3-arbres.html"><a href="3-arbres.html#phylogénie"><i class="fa fa-check"></i><b>3.3.1</b> Phylogénie</a></li>
<li class="chapter" data-level="3.3.2" data-path="3-arbres.html"><a href="3-arbres.html#taxonomie"><i class="fa fa-check"></i><b>3.3.2</b> Taxonomie</a></li>
<li class="chapter" data-level="3.3.3" data-path="3-arbres.html"><a href="3-arbres.html#cortree"><i class="fa fa-check"></i><b>3.3.3</b> Arbre des corrélations</a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="3-arbres.html"><a href="3-arbres.html#comparaison-entre-les-arbres"><i class="fa fa-check"></i><b>3.4</b> Comparaison entre les arbres</a><ul>
<li class="chapter" data-level="3.4.1" data-path="3-arbres.html"><a href="3-arbres.html#forêt-darbres"><i class="fa fa-check"></i><b>3.4.1</b> Forêt d’arbres</a></li>
<li class="chapter" data-level="3.4.2" data-path="3-arbres.html"><a href="3-arbres.html#distance-entre-les-arbres"><i class="fa fa-check"></i><b>3.4.2</b> Distance entre les arbres</a></li>
<li class="chapter" data-level="3.4.3" data-path="3-arbres.html"><a href="3-arbres.html#checktreefdr"><i class="fa fa-check"></i><b>3.4.3</b> Choix de l’arbre et lissage de z-score</a></li>
<li class="chapter" data-level="3.4.4" data-path="3-arbres.html"><a href="3-arbres.html#choix-de-larbre-et-fdr-hiérarchique"><i class="fa fa-check"></i><b>3.4.4</b> Choix de l’arbre et FDR hiérarchique</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="4-nouvelleapproche.html"><a href="4-nouvelleapproche.html"><i class="fa fa-check"></i><b>4</b> <em>zazou</em> : une nouvelle approche</a><ul>
<li class="chapter" data-level="4.1" data-path="4-nouvelleapproche.html"><a href="4-nouvelleapproche.html#ou"><i class="fa fa-check"></i><b>4.1</b> Processus d’Ornstein-Uhlenbeck</a></li>
<li class="chapter" data-level="4.2" data-path="4-nouvelleapproche.html"><a href="4-nouvelleapproche.html#zazousection"><i class="fa fa-check"></i><b>4.2</b> Zazou</a><ul>
<li class="chapter" data-level="4.2.1" data-path="4-nouvelleapproche.html"><a href="4-nouvelleapproche.html#modèle"><i class="fa fa-check"></i><b>4.2.1</b> Modèle</a></li>
<li class="chapter" data-level="4.2.2" data-path="4-nouvelleapproche.html"><a href="4-nouvelleapproche.html#estimation-ponctuelle"><i class="fa fa-check"></i><b>4.2.2</b> Estimation ponctuelle</a></li>
<li class="chapter" data-level="4.2.3" data-path="4-nouvelleapproche.html"><a href="4-nouvelleapproche.html#débiaisage-et-intervalles-de-confiance"><i class="fa fa-check"></i><b>4.2.3</b> Débiaisage et intervalles de confiance</a></li>
<li class="chapter" data-level="4.2.4" data-path="4-nouvelleapproche.html"><a href="4-nouvelleapproche.html#correction-pour-tests-multiples"><i class="fa fa-check"></i><b>4.2.4</b> Correction pour tests multiples</a></li>
</ul></li>
<li class="chapter" data-level="4.3" data-path="4-nouvelleapproche.html"><a href="4-nouvelleapproche.html#evaluation"><i class="fa fa-check"></i><b>4.3</b> Évaluation de la méthode</a><ul>
<li class="chapter" data-level="4.3.1" data-path="4-nouvelleapproche.html"><a href="4-nouvelleapproche.html#simuzazou"><i class="fa fa-check"></i><b>4.3.1</b> Données simulées</a></li>
<li class="chapter" data-level="4.3.2" data-path="4-nouvelleapproche.html"><a href="4-nouvelleapproche.html#influence-de-lâge"><i class="fa fa-check"></i><b>4.3.2</b> Influence de l’âge</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="5-ananum.html"><a href="5-ananum.html"><i class="fa fa-check"></i><b>5</b> Problèmes d’analyse numérique</a><ul>
<li class="chapter" data-level="5.1" data-path="5-ananum.html"><a href="5-ananum.html#shooting"><i class="fa fa-check"></i><b>5.1</b> Algorithme du <em>shooting</em></a></li>
<li class="chapter" data-level="5.2" data-path="5-ananum.html"><a href="5-ananum.html#mam"><i class="fa fa-check"></i><b>5.2</b> Minimisation sous contrainte de <span class="math inline">\(x^TAx\)</span></a></li>
<li class="chapter" data-level="5.3" data-path="5-ananum.html"><a href="5-ananum.html#proj"><i class="fa fa-check"></i><b>5.3</b> Projection sur un ensemble de faisabilité</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="conclusion-et-perspectives.html"><a href="conclusion-et-perspectives.html"><i class="fa fa-check"></i>Conclusion et perspectives</a><ul>
<li class="chapter" data-level="" data-path="conclusion-et-perspectives.html"><a href="conclusion-et-perspectives.html#détection-versus-prédiction"><i class="fa fa-check"></i>Détection versus prédiction</a></li>
<li class="chapter" data-level="" data-path="conclusion-et-perspectives.html"><a href="conclusion-et-perspectives.html#autres-procédures-hiérarchiques"><i class="fa fa-check"></i>Autres procédures hiérarchiques</a></li>
<li class="chapter" data-level="" data-path="conclusion-et-perspectives.html"><a href="conclusion-et-perspectives.html#amélioration-de-zazou"><i class="fa fa-check"></i>Amélioration de zazou</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="digest.html"><a href="digest.html"><i class="fa fa-check"></i>Digest</a><ul>
<li class="chapter" data-level="" data-path="digest.html"><a href="digest.html#chapter-i"><i class="fa fa-check"></i>Chapter I</a></li>
<li class="chapter" data-level="" data-path="digest.html"><a href="digest.html#chapter-ii"><i class="fa fa-check"></i>Chapter II</a></li>
<li class="chapter" data-level="" data-path="digest.html"><a href="digest.html#chapter-iii"><i class="fa fa-check"></i>Chapter III</a></li>
<li class="chapter" data-level="" data-path="digest.html"><a href="digest.html#chapter-iv"><i class="fa fa-check"></i>Chapter IV</a></li>
<li class="chapter" data-level="" data-path="digest.html"><a href="digest.html#chapter-v"><i class="fa fa-check"></i>Chapter V</a></li>
<li class="chapter" data-level="" data-path="digest.html"><a href="digest.html#conclusions-and-outlooks"><i class="fa fa-check"></i>Conclusions and outlooks</a></li>
</ul></li>
<li class="appendix"><span><b>Annexes</b></span></li>
<li class="chapter" data-level="A" data-path="A-notations.html"><a href="A-notations.html"><i class="fa fa-check"></i><b>A</b> Notations</a></li>
<li class="chapter" data-level="B" data-path="B-productions-scientifiques.html"><a href="B-productions-scientifiques.html"><i class="fa fa-check"></i><b>B</b> Productions scientifiques</a><ul>
<li class="chapter" data-level="" data-path="B-productions-scientifiques.html"><a href="B-productions-scientifiques.html#quantifying-the-impact-of-tree-choice-in-metagenomics-differential-abundance-studies-with-r"><i class="fa fa-check"></i>Quantifying the impact of tree choice in metagenomics differential abundance studies with R</a></li>
<li class="chapter" data-level="" data-path="B-productions-scientifiques.html"><a href="B-productions-scientifiques.html#incorporating-phylogenetic-information-in-microbiome-differential-abundance-studies-has-no-effect-on-detection-power-and-fdr-control"><i class="fa fa-check"></i>Incorporating Phylogenetic Information in Microbiome Differential Abundance Studies Has No Effect on Detection Power and FDR Control</a></li>
<li class="chapter" data-level="" data-path="B-productions-scientifiques.html"><a href="B-productions-scientifiques.html#hierarchical-correction-of-p-values-via-a-tree-running-ornstein-uhlenbeck-process"><i class="fa fa-check"></i>Hierarchical correction of p-values via a tree running Ornstein-Uhlenbeck process</a></li>
<li class="chapter" data-level="" data-path="B-productions-scientifiques.html"><a href="B-productions-scientifiques.html#packages-r"><i class="fa fa-check"></i>Packages R</a><ul>
<li class="chapter" data-level="" data-path="B-productions-scientifiques.html"><a href="B-productions-scientifiques.html#yatah"><i class="fa fa-check"></i>yatah</a></li>
<li class="chapter" data-level="" data-path="B-productions-scientifiques.html"><a href="B-productions-scientifiques.html#evabic"><i class="fa fa-check"></i>evabic</a></li>
<li class="chapter" data-level="" data-path="B-productions-scientifiques.html"><a href="B-productions-scientifiques.html#correlationtree"><i class="fa fa-check"></i>correlationtree</a></li>
<li class="chapter" data-level="" data-path="B-productions-scientifiques.html"><a href="B-productions-scientifiques.html#zazou"><i class="fa fa-check"></i>zazou</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="C" data-path="C-vignette.html"><a href="C-vignette.html"><i class="fa fa-check"></i><b>C</b> Vignette de zazou</a></li>
<li class="chapter" data-level="" data-path="références.html"><a href="références.html"><i class="fa fa-check"></i>Références</a></li>
<li class="chapter" data-level="" data-path="mentions-légales.html"><a href="mentions-légales.html"><i class="fa fa-check"></i>Mentions légales</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Prise en compte de l’organisation hiérarchique des espèces pour la découverte de signatures métagénomiques multi-échelles</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<script type="text/x-mathjax-config">
MathJax.Hub.Config({
  TeX: {
    Macros: {
      NN: "{\\mathbb{N}}",
      ZZ: "{\\mathbb{Z}}",
      QQ: "{\\mathbb{Q}}",
      RR: "{\\mathbb{R}}",
      shiftset: "{\\mathcal{D}}",
      dx: ["{\\mathrm{d}^{#1}\\mspace{-1mu}\\mathord{#2}}", 2, ""],
      indic: "{\\unicode{x1D7D9}}",
      prob: "\\mathop{\\mathbb{P}}",
      esp: "\\mathop{\\mathbb{E}}",
      var: "\\mathop{\\mathbb{V}\\text{ar}}",
      cov: "\\mathop{\\mathbb{C}\\text{ov}}",
      PP: ["{\\prob\\left({#1}\\right)}", 1],
      EE: ["{\\esp\\left[{#1}\\right]}", 1],
      VV: ["{\\var\\left[{#1}\\right]}", 1],
      CC: ["{\\cov\\left[{#1}\\right]}", 1],
      normal: ["{\\mathcal{N}\\left({#1},{#2}\\right)}", 2],
      ou: ["{#1}_{\\text{ou}}", 1],
      oui: ["{#1}_{\\text{ou},#2}", 2],
      pv: "{\\mathfrak{p}}",
      qv: "{\\mathfrak{q}}",
      zs: "{\\mathfrak{z}}",
      ts: "{\\mathfrak{t}}",
      sign: "{\\mathfrak{s}}",
      shifts: "{\\delta}",
      optim: "{\\beta}",
      param: "{\\theta}",
      unif: ["{\\mathcal{U}\\left({#1}\\right)}", 1],
      argmin: "\\mathop{\\mathrm{argmin}}",
      diag: "\\mathop{\\mathrm{Diag}}",
      rang: "\\mathop{\\mathrm{rang}}",
      pa: "\\mathop{\\mathrm{pa}}",
      mrca: "\\mathop{\\mathrm{mrca}}",
      desc: "\\mathop{\\mathrm{desc}}",
      warning: ["\\color{red}{{#1}}", 1]
    }
  }
});
</script>
<div id="références" class="section level1 unnumbered">
<h1>Références</h1>
<!--
This manually sets the header for this unnumbered chapter.
-->

<!--
To remove the indentation of the first entry.
-->
<p></p>
<!--
To create a hanging indent and spacing between entries.  These three lines may need to be removed for styles that don't require the hanging indent.
-->

<!--
This is just for testing with more citations for the bibliography at the end.  Add other entries into the list here if you'd like them to appear in the bibliography even if they weren't explicitly cited in the document.
-->
<div id="refs">
<div id="ref-abrahamsson2012low">
<p>Abrahamsson, T. R., Jakobsson, H. E., Andersson, A. F., Björkstén, B., Engstrand, L., &amp; Jenmalm, M. C. (2012). Low diversity of the gut microbiota in infants with atopic eczema. <em>Journal of Allergy and Clinical Immunology</em>, <em>129</em>(2), 434–440.</p>
</div>
<div id="ref-aitchison1982statistical">
<p>Aitchison, J. (1982). The statistical analysis of compositional data. <em>Journal of the Royal Statistical Society: Series B (Methodological)</em>, <em>44</em>(2), 139–160.</p>
</div>
<div id="ref-aitchison1986statistical">
<p>Aitchison, J. (1986). The statistical analysis of compositional data. Monographs on statistics and applied probability (reprinted in 2003).</p>
</div>
<div id="ref-albarede1996introduction">
<p>Albarède, F. (1996). <em>Introduction to geochemical modeling</em>. Cambridge University Press.</p>
</div>
<div id="ref-almeida2020unified">
<p>Almeida, A., Nayfach, S., Boland, M., Strozzi, F., Beracochea, M., Shi, Z. J., … others. (2020). A unified catalog of 204,938 reference genomes from the human gut microbiome. <em>Nature Biotechnology</em>, 1–10.</p>
</div>
<div id="ref-anders2010differential">
<p>Anders, S., &amp; Huber, W. (2010). Differential expression analysis for sequence count data. <em>Nature Precedings</em>, 1–1.</p>
</div>
<div id="ref-araya2020microbiome">
<p>Araya, J. P., González, M., Cardinale, M., Schnell, S., &amp; Stoll, A. (2020). Microbiome dynamics associated with the atacama flowering desert. <em>Frontiers in Microbiology</em>, <em>10</em>, 3160.</p>
</div>
<div id="ref-aronson2017specific">
<p>Aronson, H. S., Zellmer, A. J., &amp; Goffredi, S. K. (2017). The specific and exclusive microbiome of the deep-sea bone-eating snail, rubyspira osteovora. <em>FEMS Microbiology Ecology</em>, <em>93</em>(3), fiw250.</p>
</div>
<div id="ref-arumugam2011enterotypes">
<p>Arumugam, M., Raes, J., Pelletier, E., Le Paslier, D., Yamada, T., Mende, D. R., … others. (2011). Enterotypes of the human gut microbiome. <em>Nature</em>, <em>473</em>(7346), 174–180.</p>
</div>
<div id="ref-bastide2018inference">
<p>Bastide, P., Ané, C., Robin, S., &amp; Mariadassou, M. (2018). Inference of adaptive shifts for multivariate correlated traits. <em>Systematic Biology</em>, <em>67</em>(4), 662–680.</p>
</div>
<div id="ref-bastide2017detection">
<p>Bastide, P., Mariadassou, M., &amp; Robin, S. (2017). Detection of adaptive shifts on phylogenies by using shifted stochastic processes on a tree. <em>Journal of the Royal Statistical Society: Series B (Statistical Methodology)</em>, <em>79</em>(4), 1067–1093.</p>
</div>
<div id="ref-baudry2012slope">
<p>Baudry, J.-P., Maugis, C., &amp; Michel, B. (2012). Slope heuristics: Overview and implementation. <em>Statistics and Computing</em>, <em>22</em>(2), 455–470.</p>
</div>
<div id="ref-bedarf2017functional">
<p>Bedarf, J. R., Hildebrand, F., Coelho, L. P., Sunagawa, S., Bahram, M., Goeser, F., … Wüllner, U. (2017). Functional implications of microbial and viral gut metagenome changes in early stage l-dopa-naïve parkinson’s disease patients. <em>Genome Medicine</em>, <em>9</em>(1), 1–13.</p>
</div>
<div id="ref-benjamini1995controlling">
<p>Benjamini, Y., &amp; Hochberg, Y. (1995). Controlling the false discovery rate: A practical and powerful approach to multiple testing. <em>Journal of the Royal Statistical Society: Series B (Methodological)</em>, <em>57</em>(1), 289–300.</p>
</div>
<div id="ref-benjamini2001control">
<p>Benjamini, Y., &amp; Yekutieli, D. (2001). The control of the false discovery rate in multiple testing under dependency. <em>Annals of Statistics</em>, 1165–1188.</p>
</div>
<div id="ref-benoit2016multiple">
<p>Benoit, G., Peterlongo, P., Mariadassou, M., Drezen, E., Schbath, S., Lavenier, D., &amp; Lemaitre, C. (2016). Multiple comparative metagenomics using multiset k-mer counting. <em>PeerJ Computer Science</em>, <em>2</em>, e94.</p>
</div>
<div id="ref-bichat2020incorporating">
<p>Bichat, A., Plassais, J., Ambroise, C., &amp; Mariadassou, M. (2020). Incorporating phylogenetic information in microbiome differential abundance studies has no effect on detection power and fdr control. <em>Frontiers in Microbiology</em>, <em>11</em>, 649. <a href="http://doi.org/10.3389/fmicb.2020.00649">http://doi.org/10.3389/fmicb.2020.00649</a></p>
</div>
<div id="ref-billera2001geometry">
<p>Billera, L. J., Holmes, S. P., &amp; Vogtmann, K. (2001). Geometry of the space of phylogenetic trees. <em>Advances in Applied Mathematics</em>, <em>27</em>(4), 733–767.</p>
</div>
<div id="ref-blander2017regulation">
<p>Blander, J. M., Longman, R. S., Iliev, I. D., Sonnenberg, G. F., &amp; Artis, D. (2017). Regulation of inflammation by microbiota interactions with the host. <em>Nature Immunology</em>, <em>18</em>(8), 851–860.</p>
</div>
<div id="ref-bokulich2016antibiotics">
<p>Bokulich, N. A., Chung, J., Battaglia, T., Henderson, N., Jay, M., Li, H., … others. (2016). Antibiotics, birth mode, and diet shape microbiome maturation during early life. <em>Science Translational Medicine</em>, <em>8</em>(343), 343ra82–343ra82.</p>
</div>
<div id="ref-brady2009phymm">
<p>Brady, A., &amp; Salzberg, S. L. (2009). Phymm and phymmbl: Metagenomic phylogenetic classification with interpolated markov models. <em>Nature Methods</em>, <em>6</em>(9), 673–676.</p>
</div>
<div id="ref-brito2016mobile">
<p>Brito, I. L., Yilmaz, S., Huang, K., Xu, L., Jupiter, S. D., Jenkins, A. P., … others. (2016). Mobile genes in the human microbiome are structured from global to individual scales. <em>Nature</em>, <em>535</em>(7612), 435–439.</p>
</div>
<div id="ref-callahan2017exact">
<p>Callahan, B. J., McMurdie, P. J., &amp; Holmes, S. P. (2017). Exact sequence variants should replace operational taxonomic units in marker-gene data analysis. <em>The ISME Journal</em>, <em>11</em>(12), 2639–2643.</p>
</div>
<div id="ref-callahan2016dada2">
<p>Callahan, B. J., McMurdie, P. J., Rosen, M. J., Han, A. W., Johnson, A. J. A., &amp; Holmes, S. P. (2016). DADA2: High-resolution sample inference from illumina amplicon data. <em>Nature Methods</em>, <em>13</em>(7), 581.</p>
</div>
<div id="ref-canani2011potential">
<p>Canani, R. B., Di Costanzo, M., Leone, L., Pedata, M., Meli, R., &amp; Calignano, A. (2011). Potential beneficial effects of butyrate in intestinal and extraintestinal diseases. <em>World Journal of Gastroenterology: WJG</em>, <em>17</em>(12), 1519.</p>
</div>
<div id="ref-caporaso2010qiime">
<p>Caporaso, J. G., Kuczynski, J., Stombaugh, J., Bittinger, K., Bushman, F. D., Costello, E. K., … others. (2010). QIIME allows analysis of high-throughput community sequencing data. <em>Nature Methods</em>, <em>7</em>(5), 335.</p>
</div>
<div id="ref-caporaso2011global">
<p>Caporaso, J. G., Lauber, C. L., Walters, W. A., Berg-Lyons, D., Lozupone, C. A., Turnbaugh, P. J., … Knight, R. (2011). Global patterns of 16S rRNA diversity at a depth of millions of sequences per sample. <em>Proceedings of the National Academy of Sciences</em>, <em>108</em>(Supplement 1), 4516–4522.</p>
</div>
<div id="ref-cavalli1967phylogenetic">
<p>Cavalli-Sforza, L. L., &amp; Edwards, A. W. (1967). Phylogenetic analysis. Models and estimation procedures. <em>American Journal of Human Genetics</em>, <em>19</em>(3 Pt 1), 233.</p>
</div>
<div id="ref-cekanaviciute2017gut">
<p>Cekanaviciute, E., Yoo, B. B., Runia, T. F., Debelius, J. W., Singh, S., Nelson, C. A., … others. (2017). Gut bacteria from multiple sclerosis patients modulate human t cells and exacerbate symptoms in mouse models. <em>Proceedings of the National Academy of Sciences</em>, <em>114</em>(40), 10713–10718.</p>
</div>
<div id="ref-chaillou2015origin">
<p>Chaillou, S., Chaulot-Talmon, A., Caekebeke, H., Cardinal, M., Christieans, S., Denis, C., … others. (2015). Origin and ecological selection of core and food-specific bacterial communities associated with meat and seafood spoilage. <em>The ISME Journal</em>, <em>9</em>(5), 1105–1118.</p>
</div>
<div id="ref-chen2018omnibus">
<p>Chen, J., King, E., Deek, R., Wei, Z., Yu, Y., Grill, D., &amp; Ballman, K. (2018). An omnibus test for differential distribution analysis of microbiome sequencing data. <em>Bioinformatics</em>, <em>34</em>(4), 643–651.</p>
</div>
<div id="ref-chene2019microbiome">
<p>Chene, L., Sader, C. D., Magalhaes, J., Strozzi, F., Tibaldi, L., Mendez, C., … Bonny, C. (2019). Microbiome derived peptides stimulate strong immune response against tumor associated antigens and trigger in vivo tumor regression after vaccination. AACR.</p>
</div>
<div id="ref-chong2019microbiome">
<p>Chong, P. P., Chin, V. K., Looi, C. Y., Wong, W. F., Madhavan, P., &amp; Yong, V. C. (2019). The microbiome and irritable bowel syndrome–a review on the pathophysiology, current research and future therapy. <em>Frontiers in Microbiology</em>, <em>10</em>, 1136.</p>
</div>
<div id="ref-coelho2019ng">
<p>Coelho, L. P., Alves, R., Monteiro, P., Huerta-Cepas, J., Freitas, A. T., &amp; Bork, P. (2019). NG-meta-profiler: Fast processing of metagenomes using ngless, a domain-specific language. <em>Microbiome</em>, <em>7</em>(1), 84.</p>
</div>
<div id="ref-cuthbertson2014time">
<p>Cuthbertson, L., Rogers, G. B., Walker, A. W., Oliver, A., Hafiz, T., Hoffman, L. R., … Van Der Gast, C. J. (2014). Time between collection and storage significantly influences bacterial sequence composition in sputum samples from cystic fibrosis respiratory infections. <em>Journal of Clinical Microbiology</em>, <em>52</em>(8), 3011–3016.</p>
</div>
<div id="ref-david2014diet">
<p>David, L. A., Maurice, C. F., Carmody, R. N., Gootenberg, D. B., Button, J. E., Wolfe, B. E., … others. (2014). Diet rapidly and reproducibly alters the human gut microbiome. <em>Nature</em>, <em>505</em>(7484), 559–563.</p>
</div>
<div id="ref-deorowicz2015kmc">
<p>Deorowicz, S., Kokot, M., Grabowski, S., &amp; Debudaj-Grabysz, A. (2015). KMC 2: Fast and resource-frugal k-mer counting. <em>Bioinformatics</em>, <em>31</em>(10), 1569–1576.</p>
</div>
<div id="ref-desantis2006greengenes">
<p>DeSantis, T. Z., Hugenholtz, P., Larsen, N., Rojas, M., Brodie, E. L., Keller, K., … Andersen, G. L. (2006). Greengenes, a chimera-checked 16S rRNA gene database and workbench compatible with arb. <em>Applied and Environmental Microbiology</em>, <em>72</em>(7), 5069–5072.</p>
</div>
<div id="ref-ding2019selective">
<p>Ding, X., Zhang, F., Li, Q., Ting, Z., Cui, B., &amp; Li, P. (2019). Selective microbiota transplantation is effective for controlling tourette’s syndrome. <em>Gastroenterology</em>, <em>156</em>(6), S–456.</p>
</div>
<div id="ref-edgar2010search">
<p>Edgar, R. C. (2010). Search and clustering orders of magnitude faster than blast. <em>Bioinformatics</em>, <em>26</em>(19), 2460–2461.</p>
</div>
<div id="ref-edgar2016uchime2">
<p>Edgar, R. C. (2016). UCHIME2: Improved chimera prediction for amplicon sequencing. <em>BioRxiv</em>, 074252.</p>
</div>
<div id="ref-egozcue2003isometric">
<p>Egozcue, J. J., Pawlowsky-Glahn, V., Mateu-Figueras, G., &amp; Barcelo-Vidal, C. (2003). Isometric logratio transformations for compositional data analysis. <em>Mathematical Geology</em>, <em>35</em>(3), 279–300.</p>
</div>
<div id="ref-ekekezie2020understanding">
<p>Ekekezie, C., Perler, B. K., Wexler, A., Duff, C., Lillis, C. J., &amp; Kelly, C. R. (2020). Understanding the scope of do-it-yourself fecal microbiota transplant. <em>American Journal of Gastroenterology</em>, <em>115</em>(4), 603–607.</p>
</div>
<div id="ref-eloe2013impact">
<p>Eloe-Fadrosh, E. A., McArthur, M. A., Seekatz, A. M., Drabek, E. F., Rasko, D. A., Sztein, M. B., &amp; Fraser, C. M. (2013). Impact of oral typhoid vaccination on the human gut microbiota and correlations with s. Typhi-specific immunological responses. <em>PloS One</em>, <em>8</em>(4), e62026.</p>
</div>
<div id="ref-eren2014oligotyping">
<p>Eren, A. M., Borisy, G. G., Huse, S. M., &amp; Welch, J. L. M. (2014). Oligotyping analysis of the human oral microbiome. <em>Proceedings of the National Academy of Sciences</em>, <em>111</em>(28), E2875–E2884.</p>
</div>
<div id="ref-eren2013oligotyping">
<p>Eren, A. M., Maignien, L., Sul, W. J., Murphy, L. G., Grim, S. L., Morrison, H. G., &amp; Sogin, M. L. (2013). Oligotyping: Differentiating between closely related microbial taxa using 16S rRNA gene data. <em>Methods in Ecology and Evolution</em>, <em>4</em>(12), 1111–1119.</p>
</div>
<div id="ref-fan2015activation">
<p>Fan, D., Coughlin, L. A., Neubauer, M. M., Kim, J., Kim, M. S., Zhan, X., … Koh, A. Y. (2015). Activation of hif-1<span class="math inline">\(\alpha\)</span> and ll-37 by commensal bacteria inhibits candida albicans colonization. <em>Nature Medicine</em>, <em>21</em>(7), 808.</p>
</div>
<div id="ref-felsenstein1985confidence">
<p>Felsenstein, J. (1985). Confidence limits on phylogenies: An approach using the bootstrap. <em>Evolution</em>, <em>39</em>(4), 783–791.</p>
</div>
<div id="ref-fernandes2014unifying">
<p>Fernandes, A. D., Reid, J. N., Macklaim, J. M., McMurrough, T. A., Edgell, D. R., &amp; Gloor, G. B. (2014). Unifying the analysis of high-throughput sequencing datasets: Characterizing rna-seq, 16S rRNA gene sequencing and selective growth experiments by compositional data analysis. <em>Microbiome</em>, <em>2</em>(1), 15.</p>
</div>
<div id="ref-flint2012microbial">
<p>Flint, H. J., Scott, K. P., Duncan, S. H., Louis, P., &amp; Forano, E. (2012). Microbial degradation of complex carbohydrates in the gut. <em>Gut Microbes</em>, <em>3</em>(4), 289–306.</p>
</div>
<div id="ref-foster2013gut">
<p>Foster, J. A., &amp; Neufeld, K.-A. M. (2013). Gut–brain axis: How the microbiome influences anxiety and depression. <em>Trends in Neurosciences</em>, <em>36</em>(5), 305–312.</p>
</div>
<div id="ref-freckleton2003bergmann">
<p>Freckleton, R. P., Harvey, P. H., &amp; Pagel, M. (2003). Bergmann’s rule and body size in mammals. <em>The American Naturalist</em>, <em>161</em>(5), 821–825.</p>
</div>
<div id="ref-fu1998penalized">
<p>Fu, W. J. (1998). Penalized regressions: The bridge versus the lasso. <em>Journal of Computational and Graphical Statistics</em>, <em>7</em>(3), 397–416.</p>
</div>
<div id="ref-geer2010ncbi">
<p>Geer, L. Y., Marchler-Bauer, A., Geer, R. C., Han, L., He, J., He, S., … Bryant, S. H. (2010). The ncbi biosystems database. <em>Nucleic Acids Research</em>, <em>38</em>(suppl_1), D492–D496.</p>
</div>
<div id="ref-gibson2017expert">
<p>Gibson, G. R., Hutkins, R., Sanders, M. E., Prescott, S. L., Reimer, R. A., Salminen, S. J., … others. (2017). Expert consensus document: The international scientific association for probiotics and prebiotics (isapp) consensus statement on the definition and scope of prebiotics. <em>Nature Reviews Gastroenterology &amp; Hepatology</em>, <em>14</em>(8), 491.</p>
</div>
<div id="ref-gloor2017microbiome">
<p>Gloor, G. B., Macklaim, J. M., Pawlowsky-Glahn, V., &amp; Egozcue, J. J. (2017). Microbiome datasets are compositional: And this is not optional. <em>Frontiers in Microbiology</em>, <em>8</em>, 2224.</p>
</div>
<div id="ref-gloor2016compositional">
<p>Gloor, G. B., &amp; Reid, G. (2016). Compositional analysis: A valid approach to analyze microbiome high-throughput sequencing data. <em>Canadian Journal of Microbiology</em>, <em>62</em>(8), 692–703.</p>
</div>
<div id="ref-gloor2016relative">
<p>Gloor, G. B., Wu, J. R., Pawlowsky-Glahn, V., &amp; Egozcue, J. J. (2016). It’s all relative: Analyzing microbiome data as compositions. <em>Annals of Epidemiology</em>, <em>26</em>(5), 322–329.</p>
</div>
<div id="ref-gower1966some">
<p>Gower, J. C. (1966). Some distance properties of latent root and vector methods used in multivariate analysis. <em>Biometrika</em>, <em>53</em>(3-4), 325–338.</p>
</div>
<div id="ref-hehemann2010transfer">
<p>Hehemann, J.-H., Correc, G., Barbeyron, T., Helbert, W., Czjzek, M., &amp; Michel, G. (2010). Transfer of carbohydrate-active enzymes from marine bacteria to japanese gut microbiota. <em>Nature</em>, <em>464</em>(7290), 908–912.</p>
</div>
<div id="ref-holm1979simple">
<p>Holm, S. (1979). A simple sequentially rejective multiple test procedure. <em>Scandinavian Journal of Statistics</em>, 65–70.</p>
</div>
<div id="ref-holmes2012dirichlet">
<p>Holmes, I., Harris, K., &amp; Quince, C. (2012). Dirichlet multinomial mixtures: Generative models for microbial metagenomics. <em>PloS One</em>, <em>7</em>(2), e30126.</p>
</div>
<div id="ref-huang2020treeclimbr">
<p>Huang, R., Soneson, C., Germain, P.-L., Schmidt, T., Mering, C. von, &amp; Robinson, M. (2020). TreeclimbR pinpoints the data-dependent resolution of hierarchical hypotheses. <a href="http://doi.org/10.1101/2020.06.08.140608">http://doi.org/10.1101/2020.06.08.140608</a></p>
</div>
<div id="ref-jaglin2018indole">
<p>Jaglin, M., Rhimi, M., Philippe, C., Pons, N., Bruneau, A., Goustard, B., … Rabot, S. (2018). Indole, a signaling molecule produced by the gut microbiota, negatively impacts emotional behaviors in rats. <em>Frontiers in Neuroscience</em>, <em>12</em>, 216.</p>
</div>
<div id="ref-javanmard2019false">
<p>Javanmard, A., Javadi, H., &amp; others. (2019). False discovery rate control via debiased lasso. <em>Electronic Journal of Statistics</em>, <em>13</em>(1), 1212–1253.</p>
</div>
<div id="ref-javanmard2013confidence">
<p>Javanmard, A., &amp; Montanari, A. (2013). Confidence intervals and hypothesis testing for high-dimensional statistical models. In <em>Advances in neural information processing systems</em> (pp. 1187–1195).</p>
</div>
<div id="ref-javanmard2014confidence">
<p>Javanmard, A., &amp; Montanari, A. (2014). Confidence intervals and hypothesis testing for high-dimensional regression. <em>The Journal of Machine Learning Research</em>, <em>15</em>(1), 2869–2909.</p>
</div>
<div id="ref-jiang2017discrete">
<p>Jiang, L., Amir, A., Morton, J. T., Heller, R., Arias-Castro, E., &amp; Knight, R. (2017). Discrete false-discovery rate improves identification of differentially abundant microbes. <em>MSystems</em>, <em>2</em>(6).</p>
</div>
<div id="ref-jombart2017treespace">
<p>Jombart, T., Kendall, M., Almagro-Garcia, J., &amp; Colijn, C. (2017). Treespace: Statistical exploration of landscapes of phylogenetic trees. <em>Molecular Ecology Resources</em>, <em>17</em>(6), 1385–1392.</p>
</div>
<div id="ref-jousset2017less">
<p>Jousset, A., Bienhold, C., Chatzinotas, A., Gallien, L., Gobet, A., Kurm, V., … others. (2017). Where less may be more: How the rare biosphere pulls ecosystems strings. <em>The ISME Journal</em>, <em>11</em>(4), 853–862.</p>
</div>
<div id="ref-kates2020household">
<p>Kates, A. E., Jarrett, O., Skarlupka, J. H., Sethi, A., Duster, M., Watson, L., … Safdar, N. (2020). Household pet ownership and the microbial diversity of the human gut microbiota. <em>Frontiers in Cellular and Infection Microbiology</em>, <em>10</em>, 73.</p>
</div>
<div id="ref-kelly2016gut">
<p>Kelly, T. N., Bazzano, L. A., Ajami, N. J., He, H., Zhao, J., Petrosino, J. F., … He, J. (2016). Gut microbiome associates with lifetime cardiovascular disease risk profile among bogalusa heart study participants. <em>Circulation Research</em>, <em>119</em>(8), 956–964.</p>
</div>
<div id="ref-kembel2012incorporating">
<p>Kembel, S. W., Wu, M., Eisen, J. A., &amp; Green, J. L. (2012). Incorporating 16S gene copy number information improves estimates of microbial diversity and abundance. <em>PLoS Comput Biol</em>, <em>8</em>(10), e1002743.</p>
</div>
<div id="ref-khabbazian2016fast">
<p>Khabbazian, M., Kriebel, R., Rohe, K., &amp; Ané, C. (2016). Fast and accurate detection of evolutionary shifts in ornstein–uhlenbeck models. <em>Methods in Ecology and Evolution</em>, <em>7</em>(7), 811–824.</p>
</div>
<div id="ref-kim2016centrifuge">
<p>Kim, D., Song, L., Breitwieser, F. P., &amp; Salzberg, S. L. (2016). Centrifuge: Rapid and sensitive classification of metagenomic sequences. <em>Genome Research</em>, <em>26</em>(12), 1721–1729.</p>
</div>
<div id="ref-kruskal1952use">
<p>Kruskal, W. H., &amp; Wallis, W. A. (1952). Use of ranks in one-criterion variance analysis. <em>Journal of the American Statistical Association</em>, <em>47</em>(260), 583–621.</p>
</div>
<div id="ref-kultima2012mocat">
<p>Kultima, J. R., Sunagawa, S., Li, J., Chen, W., Chen, H., Mende, D. R., … others. (2012). MOCAT: A metagenomics assembly and gene prediction toolkit. <em>PloS One</em>, <em>7</em>(10), e47656.</p>
</div>
<div id="ref-ley2006ecological">
<p>Ley, R. E., Peterson, D. A., &amp; Gordon, J. I. (2006). Ecological and evolutionary forces shaping microbial diversity in the human intestine. <em>Cell</em>, <em>124</em>(4), 837–848.</p>
</div>
<div id="ref-liu2011accurate">
<p>Liu, B., Gibbons, T., Ghodsi, M., Treangen, T., &amp; Pop, M. (2011). Accurate and fast estimation of taxonomic profiles from metagenomic shotgun sequences. <em>Genome Biology</em>, <em>12</em>(1), 1–27.</p>
</div>
<div id="ref-love2014moderated">
<p>Love, M. I., Huber, W., &amp; Anders, S. (2014). Moderated estimation of fold change and dispersion for rna-seq data with deseq2. <em>Genome Biology</em>, <em>15</em>(12), 550.</p>
</div>
<div id="ref-mai2011fecal">
<p>Mai, V., Young, C. M., Ukhanova, M., Wang, X., Sun, Y., Casella, G., … others. (2011). Fecal microbiota in premature infants prior to necrotizing enterocolitis. <em>PloS One</em>, <em>6</em>(6), e20647.</p>
</div>
<div id="ref-maidak2000rdp">
<p>Maidak, B. L., Cole, J. R., Lilburn, T. G., Parker Jr, C. T., Saxman, P. R., Stredwick, J. M., … others. (2000). The rdp (ribosomal database project) continues. <em>Nucleic Acids Research</em>, <em>28</em>(1), 173–174.</p>
</div>
<div id="ref-maidak1997rdp">
<p>Maidak, B. L., Olsen, G. J., Larsen, N., Overbeek, R., McCaughey, M. J., &amp; Woese, C. R. (1997). The rdp (ribosomal database project). <em>Nucleic Acids Research</em>, <em>25</em>(1), 109–110.</p>
</div>
<div id="ref-maillet2014commet">
<p>Maillet, N., Collet, G., Vannier, T., Lavenier, D., &amp; Peterlongo, P. (2014). COMMET: Comparing and combining multiple metagenomic datasets. In <em>2014 ieee international conference on bioinformatics and biomedicine (bibm)</em> (pp. 94–98). IEEE.</p>
</div>
<div id="ref-maillet2012compareads">
<p>Maillet, N., Lemaitre, C., Chikhi, R., Lavenier, D., &amp; Peterlongo, P. (2012). Compareads: Comparing huge metagenomic experiments. In <em>BMC bioinformatics</em> (Vol. 13, p. S10). Springer.</p>
</div>
<div id="ref-mann1947test">
<p>Mann, H. B., &amp; Whitney, D. R. (1947). On a test of whether one of two random variables is stochastically larger than the other. <em>The Annals of Mathematical Statistics</em>, 50–60.</p>
</div>
<div id="ref-mathieu1996dna">
<p>Mathieu-Daudé, F., Welsh, J., Vogt, T., &amp; McClelland, M. (1996). DNA rehybridization during pcr: The ‘c o t effect’and its consequences. <em>Nucleic Acids Research</em>, <em>24</em>(11), 2080–2086.</p>
</div>
<div id="ref-mcdonald2018american">
<p>McDonald, D., Hyde, E., Debelius, J. W., Morton, J. T., Gonzalez, A., Ackermann, G., … others. (2018). American gut: An open platform for citizen science microbiome research. <em>Msystems</em>, <em>3</em>(3), e00031–18.</p>
</div>
<div id="ref-mclachlan2004finite">
<p>McLachlan, G. J., &amp; Peel, D. (2004). <em>Finite mixture models</em>. John Wiley &amp; Sons.</p>
</div>
<div id="ref-mcmurdie2014waste">
<p>McMurdie, P. J., &amp; Holmes, S. (2014). Waste not, want not: Why rarefying microbiome data is inadmissible. <em>PLoS Comput Biol</em>, <em>10</em>(4), e1003531.</p>
</div>
<div id="ref-meyerhans1990dna">
<p>Meyerhans, A., Vartanian, J.-P., &amp; Wain-Hobson, S. (1990). DNA recombination during pcr. <em>Nucleic Acids Research</em>, <em>18</em>(7), 1687–1691.</p>
</div>
<div id="ref-modolo2015urqt">
<p>Modolo, L., &amp; Lerat, E. (2015). UrQt: An efficient software for the unsupervised quality trimming of ngs data. <em>BMC Bioinformatics</em>, <em>16</em>(1), 137.</p>
</div>
<div id="ref-mohty2018odyssee">
<p>Mohty, M., Malard, F., Vekhoff, A., Lapusan, S., Isnard, F., d’Incan, E., … others. (2018). The odyssee study: Prevention of dysbiosis complications with autologous fecal microbiota transfer (fmt) in acute myeloid leukemia (aml) patients undergoing intensive treatment: Results of a prospective multicenter trial. In <em>60th annual meeting of the american-society-of-hematology (ash)</em> (Vol. 132, p. 4). AMER SOC HEMATOLOGY.</p>
</div>
<div id="ref-morgan2012human">
<p>Morgan, X. C., &amp; Huttenhower, C. (2012). Human microbiome analysis. <em>PLoS Comput Biol</em>, <em>8</em>(12), e1002808.</p>
</div>
<div id="ref-morgan2012dysfunction">
<p>Morgan, X. C., Tickle, T. L., Sokol, H., Gevers, D., Devaney, K. L., Ward, D. V., … others. (2012). Dysfunction of the intestinal microbiome in inflammatory bowel disease and treatment. <em>Genome Biology</em>, <em>13</em>(9), R79.</p>
</div>
<div id="ref-otoole2010gut">
<p>O’Toole, P. W., &amp; Claesson, M. J. (2010). Gut microbiota: Changes throughout the lifespan from infancy to elderly. <em>International Dairy Journal</em>, <em>20</em>(4), 281–291.</p>
</div>
<div id="ref-ounit2015clark">
<p>Ounit, R., Wanamaker, S., Close, T. J., &amp; Lonardi, S. (2015). CLARK: Fast and accurate classification of metagenomic and genomic sequences using discriminative k-mers. <em>BMC Genomics</em>, <em>16</em>(1), 236.</p>
</div>
<div id="ref-owen2010fast">
<p>Owen, M., &amp; Provan, J. S. (2010). A fast algorithm for computing geodesic distances in tree space. <em>IEEE/ACM Transactions on Computational Biology and Bioinformatics</em>, <em>8</em>(1), 2–13.</p>
</div>
<div id="ref-palleja2018recovery">
<p>Palleja, A., Mikkelsen, K. H., Forslund, S. K., Kashani, A., Allin, K. H., Nielsen, T., … others. (2018). Recovery of gut microbiota of healthy adults following antibiotic exposure. <em>Nature Microbiology</em>, <em>3</em>(11), 1255–1265.</p>
</div>
<div id="ref-park2007averaged">
<p>Park, M. Y., Hastie, T., &amp; Tibshirani, R. (2007). Averaged gene expressions for regression. <em>Biostatistics</em>, <em>8</em>(2), 212–227.</p>
</div>
<div id="ref-pasolli2017accessible">
<p>Pasolli, E., Schiffer, L., Manghi, P., Renson, A., Obenchain, V., Truong, D. T., … others. (2017). Accessible, curated metagenomic data through experimenthub. <em>Nature Methods</em>, <em>14</em>(11), 1023.</p>
</div>
<div id="ref-paulson2013differential">
<p>Paulson, J. N., Stine, O. C., Bravo, H. C., &amp; Pop, M. (2013). Differential abundance analysis for microbial marker-gene surveys. <em>Nature Methods</em>, <em>10</em>(12), 1200–1202.</p>
</div>
<div id="ref-pawlowsky2007lecture">
<p>Pawlowsky-Glahn, V., Egozcue, J. J., &amp; Tolosana Delgado, R. (2007). Lecture notes on compositional data analysis.</p>
</div>
<div id="ref-philippot2010ecological">
<p>Philippot, L., Andersson, S. G., Battin, T. J., Prosser, J. I., Schimel, J. P., Whitman, W. B., &amp; Hallin, S. (2010). The ecological coherence of high bacterial taxonomic ranks. <em>Nature Reviews Microbiology</em>, <em>8</em>(7), 523–529.</p>
</div>
<div id="ref-pinto2017probiotic">
<p>Pinto-Sanchez, M. I., Hall, G. B., Ghajar, K., Nardelli, A., Bolino, C., Lau, J. T., … others. (2017). Probiotic bifidobacterium longum ncc3001 reduces depression scores and alters brain activity: A pilot study in patients with irritable bowel syndrome. <em>Gastroenterology</em>, <em>153</em>(2), 448–459.</p>
</div>
<div id="ref-pistollato2016role">
<p>Pistollato, F., Sumalla Cano, S., Elio, I., Masias Vergara, M., Giampieri, F., &amp; Battino, M. (2016). Role of gut microbiota and nutrients in amyloid formation and pathogenesis of alzheimer disease. <em>Nutrition Reviews</em>, <em>74</em>(10), 624–634.</p>
</div>
<div id="ref-plaza2018mspminer">
<p>Plaza Oñate, F., Le Chatelier, E., Almeida, M., Cervino, A. C., Gauthier, F., Magoulès, F., … Wren, J. (2018). MSPminer: Abundance-based reconstitution of microbial pan-genomes from shotgun metagenomic data. <em>Bioinformatics</em>.</p>
</div>
<div id="ref-polyak1987introduction">
<p>Polyak, B. T. (1987). Introduction to optimization. Optimization software. <em>Inc., Publications Division, New York</em>, <em>1</em>.</p>
</div>
<div id="ref-pons2010meteor">
<p>Pons, N., Batto, J.-M., Kennedy, S., Almeida, M., Boumezbeur, F., Moumen, B., &amp; others. (2010). METEOR, a platform for quantitative metagenomic profiling of complex ecosystems. <em>Journées Ouvertes En Biologie, Informatique et Mathématiques Http://Www. Jobim2010. Fr/Sites/Default/Files/Presentations/27Pons. Pdf</em>.</p>
</div>
<div id="ref-price2010fasttree">
<p>Price, M. N., Dehal, P. S., &amp; Arkin, A. P. (2010). FastTree 2–approximately maximum-likelihood trees for large alignments. <em>PloS One</em>, <em>5</em>(3), e9490.</p>
</div>
<div id="ref-qin2012metagenome">
<p>Qin, J., Li, Y., Cai, Z., Li, S., Zhu, J., Zhang, F., … others. (2012). A metagenome-wide association study of gut microbiota in type 2 diabetes. <em>Nature</em>, <em>490</em>(7418), 55–60.</p>
</div>
<div id="ref-qin2014alterations">
<p>Qin, N., Yang, F., Li, A., Prifti, E., Chen, Y., Shao, L., … others. (2014). Alterations of the human gut microbiome in liver cirrhosis. <em>Nature</em>, <em>513</em>(7516), 59–64.</p>
</div>
<div id="ref-quast2012silva">
<p>Quast, C., Pruesse, E., Yilmaz, P., Gerken, J., Schweer, T., Yarza, P., … Glöckner, F. O. (2012). The silva ribosomal rna gene database project: Improved data processing and web-based tools. <em>Nucleic Acids Research</em>, <em>41</em>(D1), D590–D596.</p>
</div>
<div id="ref-quince2017shotgun">
<p>Quince, C., Walker, A. W., Simpson, J. T., Loman, N. J., &amp; Segata, N. (2017). Shotgun metagenomics, from sampling to analysis. <em>Nature Biotechnology</em>, <em>35</em>(9), 833–844.</p>
</div>
<div id="ref-ravel2011vaginal">
<p>Ravel, J., Gajer, P., Abdo, Z., Schneider, G. M., Koenig, S. S., McCulle, S. L., … others. (2011). Vaginal microbiome of reproductive-age women. <em>Proceedings of the National Academy of Sciences</em>, <em>108</em>(Supplement 1), 4680–4687.</p>
</div>
<div id="ref-r2020">
<p>R Core Team. (2020). <em>R: A language and environment for statistical computing</em>. Vienna, Austria: R Foundation for Statistical Computing. Retrieved from <a href="https://www.R-project.org/">https://www.R-project.org/</a></p>
</div>
<div id="ref-reardon2018faecal">
<p>Reardon, S. (2018). Faecal transplants could help preserve vulnerable species. <em>Nature</em>, <em>558</em>(7709), 173–175.</p>
</div>
<div id="ref-regier2019combination">
<p>Regier, Y., Komma, K., Weigel, M., Kraiczy, P., Laisi, A., Pulliainen, A. T., … Kempf, V. A. (2019). Combination of microbiome analysis and serodiagnostics to assess the risk of pathogen transmission by ticks to humans and animals in central germany. <em>Parasites &amp; Vectors</em>, <em>12</em>(1), 11.</p>
</div>
<div id="ref-reynolds2006clustering">
<p>Reynolds, A. P., Richards, G., Iglesia, B. de la, &amp; Rayward-Smith, V. J. (2006). Clustering rules: A comparison of partitioning and hierarchical clustering algorithms. <em>Journal of Mathematical Modelling and Algorithms</em>, <em>5</em>(4), 475–504.</p>
</div>
<div id="ref-robinson1981comparison">
<p>Robinson, D. F., &amp; Foulds, L. R. (1981). Comparison of phylogenetic trees. <em>Mathematical Biosciences</em>, <em>53</em>(1-2), 131–147.</p>
</div>
<div id="ref-robinson2010edger">
<p>Robinson, M. D., McCarthy, D. J., &amp; Smyth, G. K. (2010). EdgeR: A bioconductor package for differential expression analysis of digital gene expression data. <em>Bioinformatics</em>, <em>26</em>(1), 139–140.</p>
</div>
<div id="ref-robinson2007moderated">
<p>Robinson, M. D., &amp; Smyth, G. K. (2007). Moderated statistical tests for assessing differences in tag abundance. <em>Bioinformatics</em>, <em>23</em>(21), 2881–2887.</p>
</div>
<div id="ref-rowland2018gut">
<p>Rowland, I., Gibson, G., Heinken, A., Scott, K., Swann, J., Thiele, I., &amp; Tuohy, K. (2018). Gut microbiota functions: Metabolism of nutrients and other food components. <em>European Journal of Nutrition</em>, <em>57</em>(1), 1–24.</p>
</div>
<div id="ref-sakwinska2017prebiotics">
<p>Sakwinska, O., Berger, B., Zolezzi, I. S., &amp; Holbrook, J. (2017). Prebiotics for reducing the risk of obesity later in life. WO2016026684A1.</p>
</div>
<div id="ref-sankaran2014structssi">
<p>Sankaran, K., &amp; Holmes, S. (2014). StructSSI: Simultaneous and selective inference for grouped or hierarchically structured data. <em>Journal of Statistical Software</em>, <em>59</em>(13), 1.</p>
</div>
<div id="ref-schloss2009introducing">
<p>Schloss, P. D., Westcott, S. L., Ryabin, T., Hall, J. R., Hartmann, M., Hollister, E. B., … others. (2009). Introducing mothur: Open-source, platform-independent, community-supported software for describing and comparing microbial communities. <em>Applied and Environmental Microbiology</em>, <em>75</em>(23), 7537–7541.</p>
</div>
<div id="ref-schretter2018gut">
<p>Schretter, C. E., Vielmetter, J., Bartos, I., Marka, Z., Marka, S., Argade, S., &amp; Mazmanian, S. K. (2018). A gut microbial factor modulates locomotor behaviour in drosophila. <em>Nature</em>, <em>563</em>(7731), 402–406.</p>
</div>
<div id="ref-segata2012metagenomic">
<p>Segata, N., Waldron, L., Ballarini, A., Narasimhan, V., Jousson, O., &amp; Huttenhower, C. (2012). Metagenomic microbial community profiling using unique clade-specific marker genes. <em>Nature Methods</em>, <em>9</em>(8), 811–814.</p>
</div>
<div id="ref-sergeant2012high">
<p>Sergeant, M. J., Constantinidou, C., Cogan, T., Penn, C. W., &amp; Pallen, M. J. (2012). High-throughput sequencing of 16S rRNA gene amplicons: Effects of extraction procedure, primer length and annealing temperature. <em>PloS One</em>, <em>7</em>(5), e38094.</p>
</div>
<div id="ref-sharon2010commensal">
<p>Sharon, G., Segal, D., Ringo, J. M., Hefetz, A., Zilber-Rosenberg, I., &amp; Rosenberg, E. (2010). Commensal bacteria play a role in mating preference of drosophila melanogaster. <em>Proceedings of the National Academy of Sciences</em>, <em>107</em>(46), 20051–20056.</p>
</div>
<div id="ref-silverman2017phylogenetic">
<p>Silverman, J. D., Washburne, A. D., Mukherjee, S., &amp; David, L. A. (2017). A phylogenetic transform enhances analysis of compositional microbiota data. <em>Elife</em>, <em>6</em>, e21887.</p>
</div>
<div id="ref-sneath1973numerical">
<p>Sneath, P. H., Sokal, R. R., &amp; others. (1973). <em>Numerical taxonomy. The principles and practice of numerical classification.</em></p>
</div>
<div id="ref-sokal1962comparison">
<p>Sokal, R. R., &amp; Rohlf, F. J. (1962). The comparison of dendrograms by objective methods. <em>Taxon</em>, <em>11</em>(2), 33–40.</p>
</div>
<div id="ref-stoddard2015rrn">
<p>Stoddard, S. F., Smith, B. J., Hein, R., Roller, B. R., &amp; Schmidt, T. M. (2015). Rrn db: Improved tools for interpreting rRNA gene abundance in bacteria and archaea and a new foundation for future development. <em>Nucleic Acids Research</em>, <em>43</em>(D1), D593–D598.</p>
</div>
<div id="ref-stokholm2018maturation">
<p>Stokholm, J., Blaser, M. J., Thorsen, J., Rasmussen, M. A., Waage, J., Vinding, R. K., … others. (2018). Maturation of the gut microbiome and risk of asthma in childhood. <em>Nature Communications</em>, <em>9</em>(1), 1–10.</p>
</div>
<div id="ref-sun2012scaled">
<p>Sun, T., &amp; Zhang, C.-H. (2012). Scaled sparse linear regression. <em>Biometrika</em>, <em>99</em>(4), 879–898.</p>
</div>
<div id="ref-tibshirani1996regression">
<p>Tibshirani, R. (1996). Regression shrinkage and selection via the lasso. <em>Journal of the Royal Statistical Society: Series B (Methodological)</em>, <em>58</em>(1), 267–288.</p>
</div>
<div id="ref-truong2015metaphlan2">
<p>Truong, D. T., Franzosa, E. A., Tickle, T. L., Scholz, M., Weingart, G., Pasolli, E., … Segata, N. (2015). MetaPhlAn2 for enhanced metagenomic taxonomic profiling. <em>Nature Methods</em>, <em>12</em>(10), 902–903.</p>
</div>
<div id="ref-turnbaugh2009core">
<p>Turnbaugh, P. J., Hamady, M., Yatsunenko, T., Cantarel, B. L., Duncan, A., Ley, R. E., … others. (2009). A core gut microbiome in obese and lean twins. <em>Nature</em>, <em>457</em>(7228), 480–484.</p>
</div>
<div id="ref-valdez2014influence">
<p>Valdez, Y., Brown, E. M., &amp; Finlay, B. B. (2014). Influence of the microbiota on vaccine effectiveness. <em>Trends in Immunology</em>, <em>35</em>(11), 526–537.</p>
</div>
<div id="ref-vandeputte2017quantitative">
<p>Vandeputte, D., Kathagen, G., D’hoe, K., Vieira-Silva, S., Valles-Colomer, M., Sabino, J., … others. (2017). Quantitative microbiome profiling links gut community variation to microbial load. <em>Nature</em>, <em>551</em>(7681), 507–511.</p>
</div>
<div id="ref-nood2013duodenal">
<p>Van Nood, E., Vrieze, A., Nieuwdorp, M., Fuentes, S., Zoetendal, E. G., Vos, W. M. de, … others. (2013). Duodenal infusion of donor feces for recurrent clostridium difficile. <em>New England Journal of Medicine</em>, <em>368</em>(5), 407–415.</p>
</div>
<div id="ref-tidyverse2019">
<p>Wickham, H., Averick, M., Bryan, J., Chang, W., McGowan, L. D., François, R., … Yutani, H. (2019). Welcome to the tidyverse. <em>Journal of Open Source Software</em>, <em>4</em>(43), 1686. <a href="http://doi.org/10.21105/joss.01686">http://doi.org/10.21105/joss.01686</a></p>
</div>
<div id="ref-wilcoxon1992individual">
<p>Wilcoxon, F. (1992). Individual comparisons by ranking methods. In <em>Breakthroughs in statistics</em> (pp. 196–202). Springer.</p>
</div>
<div id="ref-wilgenbusch2017visualizing">
<p>Wilgenbusch, J. C., Huang, W., &amp; Gallivan, K. A. (2017). Visualizing phylogenetic tree landscapes. <em>BMC Bioinformatics</em>, <em>18</em>(1), 85.</p>
</div>
<div id="ref-wood2014kraken">
<p>Wood, D. E., &amp; Salzberg, S. L. (2014). Kraken: Ultrafast metagenomic sequence classification using exact alignments. <em>Genome Biology</em>, <em>15</em>(3), 1–12.</p>
</div>
<div id="ref-wright2012decipher">
<p>Wright, E. S., Yilmaz, L. S., &amp; Noguera, D. R. (2012). DECIPHER, a search-based approach to chimera identification for 16S rRNA sequences. <em>Applied and Environmental Microbiology</em>, <em>78</em>(3), 717–725.</p>
</div>
<div id="ref-wu2011linking">
<p>Wu, G. D., Chen, J., Hoffmann, C., Bittinger, K., Chen, Y.-Y., Keilbaugh, S. A., … others. (2011). Linking long-term dietary patterns with gut microbial enterotypes. <em>Science</em>, <em>334</em>(6052), 105–108.</p>
</div>
<div id="ref-xia2018statistical">
<p>Xia, Y., Sun, J., &amp; Chen, D.-G. (2018). <em>Statistical analysis of microbiome data with r</em>. Springer.</p>
</div>
<div id="ref-xiao2017false">
<p>Xiao, J., Cao, H., &amp; Chen, J. (2017). False discovery rate control incorporating phylogenetic tree increases detection power in microbiome-wide multiple testing. <em>Bioinformatics</em>, <em>33</em>(18), 2873–2881.</p>
</div>
<div id="ref-xinyan2016zero">
<p>Xinyan, Z., Himel, M., &amp; Nengjun, Y. (2016). Zero-inflated negative binomial regression for differential abundance testing in microbiome studies. <em>Journal of Bioinformatics and Genomics</em>, (2), 1–1.</p>
</div>
<div id="ref-yatsunenko2012human">
<p>Yatsunenko, T., Rey, F. E., Manary, M. J., Trehan, I., Dominguez-Bello, M. G., Contreras, M., … others. (2012). Human gut microbiome viewed across age and geography. <em>Nature</em>, <em>486</em>(7402), 222–227.</p>
</div>
<div id="ref-yekutieli2008hierarchical">
<p>Yekutieli, D. (2008). Hierarchical false discovery rate–controlling methodology. <em>Journal of the American Statistical Association</em>, <em>103</em>(481), 309–316.</p>
</div>
<div id="ref-yu2017ggtree">
<p>Yu, G., Smith, D. K., Zhu, H., Guan, Y., &amp; Lam, T. T.-Y. (2017). Ggtree: An r package for visualization and annotation of phylogenetic trees with their covariates and other associated data. <em>Methods in Ecology and Evolution</em>, <em>8</em>(1), 28–36.</p>
</div>
<div id="ref-zeller2014potential">
<p>Zeller, G., Tap, J., Voigt, A. Y., Sunagawa, S., Kultima, J. R., Costea, P. I., … others. (2014). Potential of fecal microbiota for early-stage detection of colorectal cancer. <em>Molecular Systems Biology</em>, <em>10</em>(11), 766.</p>
</div>
<div id="ref-zhang2014confidence">
<p>Zhang, C.-H., &amp; Zhang, S. S. (2014). Confidence intervals for low dimensional parameters in high dimensional linear models. <em>Journal of the Royal Statistical Society: Series B (Statistical Methodology)</em>, <em>76</em>(1), 217–242.</p>
</div>
<div id="ref-zhang2012should">
<p>Zhang, F., Luo, W., Shi, Y., Fan, Z., &amp; Ji, G. (2012). Should we standardize the 1,700-year-old fecal microbiota transplantation? <em>American Journal of Gastroenterology</em>, <em>107</em>(11), 1755.</p>
</div>
<div id="ref-zhang2017negative">
<p>Zhang, X., Mallick, H., Tang, Z., Zhang, L., Cui, X., Benson, A. K., &amp; Yi, N. (2017). Negative binomial mixed models for analyzing microbiome count data. <em>BMC Bioinformatics</em>, <em>18</em>(1), 4.</p>
</div>
<div id="ref-zheng2019gut">
<p>Zheng, P., Zeng, B., Liu, M., Chen, J., Pan, J., Han, Y., … others. (2019). The gut microbiome from patients with schizophrenia modulates the glutamate-glutamine-gaba cycle and schizophrenia-relevant behaviors in mice. <em>Science Advances</em>, <em>5</em>(2), eaau8317.</p>
</div>
<div id="ref-zhernakova2016population">
<p>Zhernakova, A., Kurilshikov, A., Bonder, M. J., Tigchelaar, E. F., Schirmer, M., Vatanen, T., … others. (2016). Population-based metagenomics analysis reveals markers for gut microbiome composition and diversity. <em>Science</em>, <em>352</em>(6285), 565–569.</p>
</div>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="C-vignette.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="mentions-légales.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": true,
"facebook": false,
"twitter": false,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["twitter", "linkedin", "facebook", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "night",
"family": "serif",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": [["thesis.pdf", "PDF"]],
"toc": {
"collapse": "section"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
